\documentclass[11pt,]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage[margin=1.0in]{geometry}
\usepackage{hyperref}
\hypersetup{unicode=true,
            pdftitle={Revisiting Short-Chain Fatty Acids and the Microbiota in Colorectal Cancer},
            pdfborder={0 0 0},
            breaklinks=true}
\urlstyle{same}  % don't use monospace font for urls
\usepackage{graphicx,grffile}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
\IfFileExists{parskip.sty}{%
\usepackage{parskip}
}{% else
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{0}
% Redefines (sub)paragraphs to behave more like sections
\ifx\paragraph\undefined\else
\let\oldparagraph\paragraph
\renewcommand{\paragraph}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
\let\oldsubparagraph\subparagraph
\renewcommand{\subparagraph}[1]{\oldsubparagraph{#1}\mbox{}}
\fi

%%% Use protect on footnotes to avoid problems with footnotes in titles
\let\rmarkdownfootnote\footnote%
\def\footnote{\protect\rmarkdownfootnote}

%%% Change title format to be more compact
\usepackage{titling}

% Create subtitle command for use in maketitle
\newcommand{\subtitle}[1]{
  \posttitle{
    \begin{center}\large#1\end{center}
    }
}

\setlength{\droptitle}{-2em}
  \title{Revisiting Short-Chain Fatty Acids and the Microbiota in Colorectal
Cancer}
  \pretitle{\vspace{\droptitle}\centering\huge}
  \posttitle{\par}
  \author{}
  \preauthor{}\postauthor{}
  \date{}
  \predate{}\postdate{}

\usepackage{helvet} % Helvetica font
\renewcommand*\familydefault{\sfdefault} % Use the sans serif version of the font
\usepackage[T1]{fontenc}

\usepackage[none]{hyphenat}

\usepackage{setspace}
\doublespacing
\setlength{\parskip}{1em}

\usepackage{lineno}

\usepackage{pdfpages}

\begin{document}
\maketitle

\vspace{35mm}

Running title: SCFAs and Colorectal Cancer

\vspace{35mm}

Marc A Sze\({^1}\), Nicholas A Lesniak\({^1}\), Mack T Ruffin
IV\({^2}\), Patrick D. Schloss\({^1}\)\({^\dagger}\)

\vspace{40mm}

\(\dagger\) To whom correspondence should be addressed:
\href{mailto:pschloss@umich.edu}{\nolinkurl{pschloss@umich.edu}}

\(1\) Department of Microbiology and Immunology, University of Michigan,
Ann Arbor, MI 48109

\(2\) Department of Family Medicine and Community Medicine, Penn State
Hershey Medical Center, Hershey, PA

\newpage

\linenumbers

\subsection{Abstract}\label{abstract}

\newpage

\subsection{Introduction}\label{introduction}

\newpage

\subsection{Results}\label{results}

\newpage

\subsection{Discussion}\label{discussion}

\newpage

\subsection{Conclusions}\label{conclusions}

\newpage

\subsection{Materials and Methods}\label{materials-and-methods}

\textbf{Study design and sampling.} The overall protocol has been
described in detail previously (1, 2). In brief, this study used fecal
samples obtained at either a single cross-sectional time point (n=490)
or from before and after treatment for their tumor (n=67). For patients
undergoing treatment for their tumor the length of time between their
initial and follow up sample ranged from 188 - 546 days. Our use of
treatment has been previously defined as encompassing removal of a tumor
with or without chemotherapy and radiation (1). Diagnosis of tumor was
made by colonoscopic examination and histopathological review of
biopsies obtained (1, 2). The University of Michigan Institutional
Review Board approved the study and informed consent was obtained from
all participants in accordance to the guidelines set out by the Helsinki
Decleration.

\textbf{Measuring specific SCFAs.} Our protocol for the measurement of
acetate, butyrate, and propionate followed a previously published
protocol (3). The following changes to this protocol included the use of
frozen fecal samples suspended in 1ml of PBS instead of fecal
suspensions in DNA Genotek OmniGut tubes, and the use of the acutal
weight of fecal samples instead of the average weight for SCFA
concentration normalizations. These changes did not affect the overall
median concentrations of these SCFAs between the two studies (see Table
1 (3) and Figure 1 in this report).

\textbf{16s rRNA gene sequencing.} The workflow and processing have been
described previously (1, 4, 5). The major differences from these
previous reports include: the use of version 1.39.5 of the mothur
software package and clustering Operational Taxonomic Units (OTUs) at
97\% similarity used the OptClust algorithm (6).

\textbf{Generating imputed metagenomes.} The use of PICRUSt version
1.1.2 with the recommended standard operating protocol (7) was used.
Briefly, the mothur shared file and metadata was converted into a biom
formated table using the biom convert function, the subsequent biom file
was processed with the normalize\_by\_copy\_number.py function, and
subsequent imputed metagenomes created using the predict\_metagenomes.py
function.

\textbf{Obtaining OPFs from metagenomes.} A subset of the
cross-sectional group (n=490) containing a total of 85 individuals
(normal n=29 normal, adenoma n=28, and carcinoma n=28) was shotgun
sequenced on an Illumina HiSeq with 125 bp paired end reads using a
previously described method (8). Briefly, the sequences were quality
filtered and sequences aligning to the human genome were removed prior
to contig assembly with MEGAHIT (9). Open Reading Frames (ORFs) were
identified using Prodigal (10), counts generated using Diamond (11),
subsequent clustering into Operational Protein Families (OPFs) used
mmseq2 (12), and OPF gene alignment used the KEGG database (13).

\textbf{Pulling genes involved with SCFA synthesis.} Specific genes
located near the end of the pathways involved in the synthesis of
acetate, butyrate, and propionate were analyzed for any differences
between individuals with normal colons and those with tumors. These
genes were based on pathways from KEGG and can be found in the
supplemental material {[}Table S1{]}.

\textbf{Statistical analysis workflow.} All analysis was performed using
the statistical language R (14). Generally, differences between the
different disease groups used a Kruskal-Walis rank sum test with a
Dunn's post-hoc test. Models to classify high or low SCFA concentration
based on 16S rRNA gene sequencing data were created using Random Forest
(15). Regression models to classify the exact SCFA concentration based
on 16S rRNA gene sequencing data also were built using the Random Forest
algorithm. The measured SCFA concentrations were first tested for
differences between groups. The ability of 16S rRNA gene sequencing to
classify these concentrations were then assessed. Next, the imputed gene
counts of important mediators of SCFA creation were tested. Finally, the
counts generated for OPFs that matched important genes involved with
SCFA creation were analyzed.

\newpage

\subsection{Acknowledgements}\label{acknowledgements}

The authors thank the Great Lakes-New England Early Detection Research
Network for providing the fecal samples that were used in this study. We
would also like to thank Kwi Kim and Thomas M Schmidt for their help in
running the short-chian fatty acid analysis on the High-Performace
Liquid Chromatography machine at the University of Michigan. Salary
support for Marc A. Sze came from the Canadian Institute of Health
Research and NIH grant UL1TR002240. Salary support for Patrick D.
Schloss came from NIH grants P30DK034933 and 1R01CA215574.

\newpage

\subsection{References}\label{references}

\hypertarget{refs}{}
\hypertarget{ref-normalization_sze2017}{}
1. \textbf{Sze MA}, \textbf{Baxter NT}, \textbf{Ruffin MT},
\textbf{Rogers MAM}, \textbf{Schloss PD}. 2017. Normalization of the
microbiota in patients after treatment for colonic lesions. Microbiome
\textbf{5}.
doi:\href{https://doi.org/10.1186/s40168-017-0366-3}{10.1186/s40168-017-0366-3}.

\hypertarget{ref-crc_model_baxter2016}{}
2. \textbf{Baxter NT}, \textbf{Ruffin MT}, \textbf{Rogers MAM},
\textbf{Schloss PD}. 2016. Microbiota-based model improves the
sensitivity of fecal immunochemical test for detecting colonic lesions.
Genome Medicine \textbf{8}.
doi:\href{https://doi.org/10.1186/s13073-016-0290-3}{10.1186/s13073-016-0290-3}.

\hypertarget{ref-scfa_measures_venkataraman2016}{}
3. \textbf{Venkataraman A}, \textbf{Sieber JR}, \textbf{Schmidt AW},
\textbf{Waldron C}, \textbf{Theis KR}, \textbf{Schmidt TM}. 2016.
Variable responses of human microbiomes to dietary supplementation with
resistant starch. Microbiome \textbf{4}.
doi:\href{https://doi.org/10.1186/s40168-016-0178-x}{10.1186/s40168-016-0178-x}.

\hypertarget{ref-Schloss2009}{}
4. \textbf{Schloss PD}, \textbf{Westcott SL}, \textbf{Ryabin T},
\textbf{Hall JR}, \textbf{Hartmann M}, \textbf{Hollister EB},
\textbf{Lesniewski RA}, \textbf{Oakley BB}, \textbf{Parks DH},
\textbf{Robinson CJ}, \textbf{Sahl JW}, \textbf{Stres B},
\textbf{Thallinger GG}, \textbf{Horn DJV}, \textbf{Weber CF}. 2009.
Introducing mothur: Open-source, platform-independent,
community-supported software for describing and comparing microbial
communities. Applied and Environmental Microbiology
\textbf{75}:7537--7541.
doi:\href{https://doi.org/10.1128/aem.01541-09}{10.1128/aem.01541-09}.

\hypertarget{ref-Kozich2013}{}
5. \textbf{Kozich JJ}, \textbf{Westcott SL}, \textbf{Baxter NT},
\textbf{Highlander SK}, \textbf{Schloss PD}. 2013. Development of a
dual-index sequencing strategy and curation pipeline for analyzing
amplicon sequence data on the MiSeq illumina sequencing platform.
Applied and Environmental Microbiology \textbf{79}:5112--5120.
doi:\href{https://doi.org/10.1128/aem.01043-13}{10.1128/aem.01043-13}.

\hypertarget{ref-opticlust_westcott2017}{}
6. \textbf{Westcott SL}, \textbf{Schloss PD}. 2017. OptiClust, an
improved method for assigning amplicon-based sequence data to
operational taxonomic units. mSphere \textbf{2}:e00073--17.
doi:\href{https://doi.org/10.1128/mspheredirect.00073-17}{10.1128/mspheredirect.00073-17}.

\hypertarget{ref-Langille2013}{}
7. \textbf{Langille MGI}, \textbf{Zaneveld J}, \textbf{Caporaso JG},
\textbf{McDonald D}, \textbf{Knights D}, \textbf{Reyes JA},
\textbf{Clemente JC}, \textbf{Burkepile DE}, \textbf{Thurber RLV},
\textbf{Knight R}, \textbf{Beiko RG}, \textbf{Huttenhower C}. 2013.
Predictive functional profiling of microbial communities using 16S rRNA
marker gene sequences. Nature Biotechnology \textbf{31}:814--821.
doi:\href{https://doi.org/10.1038/nbt.2676}{10.1038/nbt.2676}.

\hypertarget{ref-Hannigan2017}{}
8. \textbf{Hannigan GD}, \textbf{Duhaime MB}, \textbf{Ruffin MT},
\textbf{Koumpouras CC}, \textbf{Schloss PD}. 2017. Diagnostic potential
\& the interactive dynamics of the colorectal cancer virome.
doi:\href{https://doi.org/10.1101/152868}{10.1101/152868}.

\hypertarget{ref-Li2015}{}
9. \textbf{Li D}, \textbf{Liu C-M}, \textbf{Luo R}, \textbf{Sadakane K},
\textbf{Lam T-W}. 2015. MEGAHIT: An ultra-fast single-node solution for
large and complex metagenomics assembly via succinct de bruijn graph.
Bioinformatics \textbf{31}:1674--1676.
doi:\href{https://doi.org/10.1093/bioinformatics/btv033}{10.1093/bioinformatics/btv033}.

\hypertarget{ref-Hyatt2010}{}
10. \textbf{Hyatt D}, \textbf{Chen G-L}, \textbf{LoCascio PF},
\textbf{Land ML}, \textbf{Larimer FW}, \textbf{Hauser LJ}. 2010.
Prodigal: Prokaryotic gene recognition and translation initiation site
identification. BMC Bioinformatics \textbf{11}:119.
doi:\href{https://doi.org/10.1186/1471-2105-11-119}{10.1186/1471-2105-11-119}.

\hypertarget{ref-Buchfink2014}{}
11. \textbf{Buchfink B}, \textbf{Xie C}, \textbf{Huson DH}. 2014. Fast
and sensitive protein alignment using DIAMOND. Nature Methods
\textbf{12}:59--60.
doi:\href{https://doi.org/10.1038/nmeth.3176}{10.1038/nmeth.3176}.

\hypertarget{ref-Steinegger2017}{}
12. \textbf{Steinegger M}, \textbf{Söding J}. 2017. MMseqs2 enables
sensitive protein sequence searching for the analysis of massive data
sets. Nature Biotechnology.
doi:\href{https://doi.org/10.1038/nbt.3988}{10.1038/nbt.3988}.

\hypertarget{ref-Kanehisa2015}{}
13. \textbf{Kanehisa M}, \textbf{Sato Y}, \textbf{Kawashima M},
\textbf{Furumichi M}, \textbf{Tanabe M}. 2015. KEGG as a reference
resource for gene and protein annotation. Nucleic Acids Research
\textbf{44}:D457--D462.
doi:\href{https://doi.org/10.1093/nar/gkv1070}{10.1093/nar/gkv1070}.

\hypertarget{ref-r_citation_2017}{}
14. \textbf{R Core Team}. 2017. R: A language and environment for
statistical computing. R Foundation for Statistical Computing, Vienna,
Austria.

\hypertarget{ref-randomforest_citation_2002}{}
15. \textbf{Liaw A}, \textbf{Wiener M}. 2002. Classification and
regression by randomForest. R News \textbf{2}:18--22.

\newpage

Insert figure legends with the first sentence in bold, for example:

\textbf{Figure 1. Number of OTUs sampled among bacterial and archaeal
16S rRNA gene sequences for different OTU definitions and level of
sequencing effort.} Rarefaction curves for different OTU definitions of
Bacteria (A) and Archaea (B). Rarefaction curves for the coarse
environments in Table 1 for Bacteria (C) and Archaea (D).

\newpage


\end{document}
