\documentclass[11pt,]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage[margin=1.0in]{geometry}
\usepackage{hyperref}
\hypersetup{unicode=true,
            pdftitle={Revisiting the Relationship between Short-Chain Fatty Acids, the Microbiota, and Colorectal Tumors},
            pdfborder={0 0 0},
            breaklinks=true}
\urlstyle{same}  % don't use monospace font for urls
\usepackage{graphicx,grffile}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
\IfFileExists{parskip.sty}{%
\usepackage{parskip}
}{% else
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{0}
% Redefines (sub)paragraphs to behave more like sections
\ifx\paragraph\undefined\else
\let\oldparagraph\paragraph
\renewcommand{\paragraph}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
\let\oldsubparagraph\subparagraph
\renewcommand{\subparagraph}[1]{\oldsubparagraph{#1}\mbox{}}
\fi

%%% Use protect on footnotes to avoid problems with footnotes in titles
\let\rmarkdownfootnote\footnote%
\def\footnote{\protect\rmarkdownfootnote}

%%% Change title format to be more compact
\usepackage{titling}

% Create subtitle command for use in maketitle
\newcommand{\subtitle}[1]{
  \posttitle{
    \begin{center}\large#1\end{center}
    }
}

\setlength{\droptitle}{-2em}
  \title{Revisiting the Relationship between Short-Chain Fatty Acids, the
Microbiota, and Colorectal Tumors}
  \pretitle{\vspace{\droptitle}\centering\huge}
  \posttitle{\par}
  \author{}
  \preauthor{}\postauthor{}
  \date{}
  \predate{}\postdate{}

\usepackage{helvet} % Helvetica font
\renewcommand*\familydefault{\sfdefault} % Use the sans serif version of the font
\usepackage[T1]{fontenc}

\usepackage[none]{hyphenat}

\usepackage{setspace}
\doublespacing
\setlength{\parskip}{1em}

\usepackage{lineno}

\usepackage{pdfpages}

\begin{document}
\maketitle

\vspace{35mm}

Running title: SCFAs and colorectal tumors

\vspace{35mm}

Marc A Sze\({^1}\), Nicholas A Lesniak\({^1}\), Mack T Ruffin
IV\({^2}\), Patrick D. Schloss\({^1}\)\({^\dagger}\)

\vspace{40mm}

\(\dagger\) To whom correspondence should be addressed:
\href{mailto:pschloss@umich.edu}{\nolinkurl{pschloss@umich.edu}}

\(1\) Department of Microbiology and Immunology, University of Michigan,
Ann Arbor, MI 48109

\(2\) Department of Family Medicine and Community Medicine, Penn State
Hershey Medical Center, Hershey, PA

\newpage

\linenumbers

\subsection{Abstract}\label{abstract}

\textbf{Background.} Colorectal cancer (CRC) is a growing health concern
with the majority of the risk for developing disease being due to
environmental factors. The microbiota is one of these environmental
factors with certain bacterial community members being associated with
CRC, while other taxa are associated to colons without tumors. Some of
the taxa associated to colons without tumors can use fiber to produce
short-chain fatty acids (SCFAs) that can inhibit tumor growth in model
systems. However, the data supporting the importance of SCFAs in human
CRC is less certain. Here, we test the hypothesis that SCFA
concentrations are different in individuals with colorectal tumors.

\textbf{Methods.} We analyzed a cross-sectional (n=490) and longitudinal
pre- and post-treatment (n=67) group for their concentrations of
acetate, butyrate, and propionate. Analysis also included tumor
classification models using Random Forest, imputed gene relative
abundance with PICRUSt, and metagenomic sequencing on a subset (n=85) of
the total cross-sectional group.

\textbf{Results.} No difference in SCFA concentrations were found
between individuals without tumors and patients with adenomas or
carcinomas (P-value \textgreater{} 0.15). There was no difference in
classification models with or without SCFAs in their ability to predict
patients with adenomas or carcinomas versus individuals without tumors
(P-value \textgreater{} 0.05). Using metagenomic sequencing, there was
also no difference in genes involved with SCFA synthesis between
individuals without tumors and patients with adenomas or carcinomas
(P-value \textgreater{} 0.70).

\textbf{Conclusions.} Although our data does not support the hypothesis
that SCFAs are different in individuals that have colorectal tumors,
there may be context specific scenarios where SCFAs may still be
beneficial for treatment of CRC. Alternatively, there may be other
mechanisms that have not been thoroughly investigated that are more
important to the development of human CRC.

\newpage

\subsection{Introduction}\label{introduction}

Colorectal cancer (CRC) is currently the third leading cancer-related
cause of death within the US (1, 2). Although there is a genetic
component to the disease, the environment has been attributed to being a
larger risk factor for CRC (3). These environmental risk factors include
but are not limited to smoking cigarettes, diet, and the microbiota
(4--6). Many of these environmental risk factors, including the
microbiota, could be easily modifiable, and this has lead to the
investigation of how the microbiota may exacerbate or cause tumorigensis
(7--9) and whether the bacterial community is altered in CRC (10, 11).
Many of these previous case/control studies have identified resident
bacterial taxa to be decreased in patients with carcinoma tumors
(11--13). Many of the bacterial species from these resident taxa
actively produce short-chain fatty acids (SCFAs) from fiber that are
part of our general diet (14). The most extensively studied of these
SCFAs are acetate, butyrate, and propionate (15). Ultimately, these
SCFAs could be the main metabolites involved with protection against
tumorigenesis and could help to reduce the risk of CRC.

Prior research suggests that SCFAs have promise in acting as an
anti-tumorigenic agent. Specific SCFAs, like butyrate, have shown
positive results within model systems (16). For example, butyrate has
been shown to inhibit cancer cell growth in \emph{in vitro} systems
(17). Additionally, fiber supplementation in mouse models of CRC caused
an overall reduction in tumor burden while also increasing SCFA
concentrations (18). These exciting results in model systems suggests
that supplementation with food sources that bacteria use to create these
SCFAs may be able to confer beneficial effects against CRC. However, it
is important to note that these model systems provide only preliminary
evidence towards the ability of SCFAs to reduce and treat tumors and the
studies reporting benefit in humans has been less convincing.

Overall, there is a lack of evidence on the benefit of increasing SCFA
concentrations to protect against CRC in human populations. The initial
case/control studies that investigated SCFA concentrations in CRC found
that patients with carcinomas had lower concentrations of acetate,
butyrate, and propionate versus patients with adenomas or individuals
without colon tumors (19). Although this would argue that increasing
SCFA concentrations could be protective against tumorigenesis, fiber
supplementation in randomized controlled trials have consistently failed
to protect against tumor recurrence (20). These findings would argue
against the utility of treatments that aim to use SCFAs to reduce or
protect against tumorigenesis. Given the lack of clear evidence in human
studies of the benefit of SCFAs in CRC, there is a need for more
investigation into this area.

Our study fills some of the current gaps in the literature that relate
to the study of SCFAs and CRC in human populations. Specifically, it
tests previous case/control findings on SCFA concentrations in
individuals with and without tumors. It also tests previous suggestions
that there is a continuous reduction in SCFA concentrations as tumor
severity worsens by increasing the number of patients with adenomas
within the study. Additionally, we build upon these observations by
assessing the utility of using SCFAs and Operational Taxonomic Units
(OTUs) as a risk stratification tool of colorectal tumors (adenoma or
carcinoma). We also investigate whether OTUs that are most important to
these models are closely associated with the classification of SCFA
concentrations. Collectively, this study provides important information
on the replicability of previous findings in humans by extensively
studying how SCFAs are associated with colorectal tumors.

To accomplish this task we directly measured the concentration of
acetate, butyrate, and propionate within fecal samples for two different
groups. The first group had a sample obtained at a single cross
sectional point in time while the second group had samples obtained
before (pre-) and after (post-)treatment for colorectal tumors. We also
used PICRUSt (21) and metagenomic sequencing to investigate if there
were any differences in genes involved with SCFA synthesis between
individuals without tumors, patients with adenomas, and patients with
carcinomas. Next, using the cross-sectional data, we analyzed the number
of correlations between OTU relative abundance and SCFA concentrations
across individuals without tumors and patients with adenomas or
carcinomas. Additionally, we assessed the affect adding SCFA
concentrations to OTU data had on classification of patients with
adenomas or carcinomas using the Random Forest algorithm (22). We also
analyzed how well 16S rRNA gene sequencing predicts SCFA concentrations.
Collectively, this investigation provides additional information as to
whether SCFAs are decreased in patients with colorectal tumors and
provides context as to whether targeting taxa to increase SCFA
concentrations is a viable option to protect against colon
tumorigenesis.

\newpage

\subsection{Results}\label{results}

\textbf{Decreased SCFA concentrations are not associated with tumors.}
We used high-performance liquid chromatography (HPLC) to measure
acetate, butyrate, and propionate concentrations of frozen fecal samples
from 490 individuals at a cross-sectional point in time. There was no
difference between individuals without colon tumors (n=172) and patients
with either an adenoma (n=198) or carcinoma (n=120) for any of the SCFAs
measured after multiple comparison correction (P-value \textgreater{}
0.15) {[}Figure 1A - 1C{]}. We next measured the concentration of SCFAs
in 67 patients with an adenoma (n=41) or carcinoma (n=26) in which we
had pre- and post-treatment fecal samples. Although there was a general
trend for increasing acetate, butyrate, and propionate concentrations
after treatment for tumors, there was no significant difference pre- and
post-treatment for patients with adenomas (P-value \textgreater{} 0.20)
or carcinomas (P-value \textgreater{} 0.80) {[}Figure 1D - 1F{]}.

\textbf{Changes in genes for enzymes involved in SCFA synthesis are not
associated with tumors.} Using a list of specific genes that are
important for the synthesis of SCFAs {[}Table S1{]}, we looked for
differences in gene abundance between individuals without colon tumors
and patients with adenomas or carcinomas. First, using imputed gene
relative abundance based on 16S rRNA gene sequencing we found no
difference in any of the genes involved with acetate, butyrate, or
propionate synthesis (P-value \textgreater{} 0.90) {[}Table S2{]}. This
similarity between groups is highlighted by visualizing genes important
in butyrate synthesis {[}Figure 2A{]}. Using a paired Wilcoxon rank-sum
test, there also was no difference in imputed gene relative abundance
between pre- and post-treatment samples for any genes involved with SCFA
synthesis (P-value \textgreater{} 0.70) {[}Table S3{]}. Next, we took a
subset of these 490 fecal samples (n=85) and used metagenomic sequencing
to confirm these results. Like the imputed gene results, metagenomic
analysis found that there was no difference in any of the genes involved
in SCFA synthesis between individuals without colon tumors (n=29) and
patients with adenoma (n=28) or carcinoma (n=28) (P-value \textgreater{}
0.70) {[}Table S4{]}. This lack of difference can be highlighted when we
visualize the results for butyrate kinase {[}Figure 2B{]}. These
observations provide evidence that gene prevalence for enzymes involved
in SCFA synthesis does not change due to colorectal tumors.

\textbf{Total significant positive correlations between OTU relative
abundance and SCFA concentration were similar for individuals without
tumors and patients with adenomas or carcinomas.} Having found no
difference between individuals without tumors and patients with adenomas
or carcinomas in SCFA concentrations or genes coding for enzymes
involved with SCFA synthesis, we next investigated if specific OTUs
correlated with SCFA concentrations. Using Spearmans rho we found that
the majority of significant OTU correlations were to taxa from
\emph{Clostridiales}, \emph{Lachnospiraceae}, and \emph{Ruminococcaceae}
{[}Figure 3 \& Table S5{]}. A similar pattern was observed when using
high/low SCFA groups based on the overall median concentration for that
specific SCFA {[}Figure S1 \& Table S6{]}. There was a noticeablely
higher number of significant negative correlations associated with
patients with adenomas for all SCFAs tested {[}Figure 3{]}. In
particular, OTUs from the \emph{Ruminococcaceae} family had the largest
share of these negative correlations within patients with adenomas
{[}Figure 3{]}. Although patients with adenomas had more positive
correlations between OTUs and SCFA concentrations, their total number
was more similar to individuals without tumors or patients with
carcinomas versus the analogous comparison for the number of negative
correlations {[}Figure 3{]}. The number of positive correlations between
OTUs and SCFA concentrations was similar between individuals without
tumors and patients with a carcinoma {[}Figure 3{]}. Overall, these
results suggest that the resident taxa that may change the most due to
colon tumors may not be ones that are responsible for the production of
acetate, butyrate, or propionate.

\textbf{SCFA concentrations do not replace important
\emph{Clostridiales}, \emph{Lachnospiraceae}, and \emph{Ruminococcaceae}
OTUs in Random Forest models built to classify tumors.} SCFA
concentrations could improve prediction of tumors based on specific
bacterial community structures. Our OTU data can be used in combination
with SCFAs to assess whether there is a community dependent context to
SCFA classification of tumors. Using the Random Forest algorithm we
built models with OTU abundance data or OTU abundances and SCFA
concentrations to classify normal versus adenoma and normal versus
carcinoma fecal samples. For adenoma and carcinoma models, there was no
difference between the median AUC of models with or without SCFA
concentrations (P-value \textgreater{} 0.05) {[}Figure 4A \& 4D{]}.
There was little difference between the top 10 most important OTUs, as
measured by mean decrease in accuracy, in models using SCFA and OTUs
versus OTUs only {[}Figure 4B-C \& 4E-F{]}. The main reason for this was
because SCFA concentrations replaced the information gained by specific
OTUs rather than add new information to the model {[}Figure 4B-C{]}. The
SCFA concentrations also only replaced select OTUs with taxonomic
classification to \emph{Clostridiales}, \emph{Lachnospiraceae}, and
\emph{Ruminococcaceae} with many OTUs within these taxa remaining in the
top 10 {[}Figure 4B \& 4C{]}. In combination with the previous results
on taxa correlations, these observations provide additional evidence
that the resident taxa within models that classify tumor are not ones
associated with acetate, butyrate, or propionate production.

\textbf{Random Forest models for SCFA concentrations have different top
10 most important OTUs than tumor models.} Using OTU data we built
Random Forest models to classify either SCFA concentration or
higher/lower than median SCFA concentrations. Overall, OTU data had a
reasonable ability to classify both SCFA concentrations and high/low
SCFA groups {[}Figure 5A \& S2A{]}. However, these models tended to be
over fit, suggesting that rarer taxa may be important for this
classification {[}Figure 5A \& S2A{]}. Additionally, there were
differences in accuracy for both model types based on whether the
individual had no tumors, an adenoma, or a carcinoma {[}Figure 5B \&
S2B{]}. There also was minimal overlap between these SCFA classification
model's most important OTUs and those used to classify patients with
adenoma or carcinoma tumors {[}Figure 4B-C, 4E-F, 5C-E, and S2C-E{]}.
The only OTU that did overlap between the models was OTU00167
(\emph{Clostridiales}) {[}Figure 4B-C, 4E-F, 5C-E, and S2C-E{]}.
Additionally, OTU00167 was in the top 10 most important OTUs for the OTU
adenoma model but not in the SCFA and OTU adenoma model while acetate
and butyrate concentrations were {[}Figure 4B-C{]}. These observations
provide further evidence that it is possible to identify specific OTUs
associated with higher SCFA concentrations and that these OTUs belong to
taxa known to produce acetate, butyrate, and propionate. Although it is
possible to identify OTUs associated with SCFA production, our results
do not support the hypothesis that SCFA concentration or OTUs associated
with their production are different between individuals with no tumors
and patients with adenomas or carcinomas.

\newpage

\subsection{Discussion}\label{discussion}

The observations from this study do not support the hypothesis that SCFA
concentrations are different in individuals with tumors. Whether we
directly measured the SCFA concentration or investigated genes
associated with their production, no difference could be identified
between individuals without tumors and patients with adenomas or
carcinomas {[}Figure 1 \& 2{]}. Although there were differences in the
number of significant correlations between SCFA concentration and OTU
relative abundance based on whether individuals did not have tumors, had
an adenoma, or had a carcinoma, SCFA concentrations did not provide
increased model accuracy for tumor classification {[}Figure 3-4 \&
S1{]}. Instead, SCFA concentrations provided similar information to what
specific OTUs were already providing to the tumor classification models
{[}Figure 4{]}. Additionally, when models using OTU relative abundance
to classify SCFA concentrations were assessed, the OTUs that classified
to \emph{Clostridiales}, \emph{Lachnospiraceae}, and
\emph{Ruminococcaceae} were not the same as the OTUs that classified to
these taxa in the tumor models {[}Figure 4-5{]}. Collectively, our
observations suggest that resident taxa from \emph{Clostridiales},
\emph{Lachnospiraceae}, and \emph{Ruminococcaceae}, that are different
between individuals without tumors and patients with adenomas or
carinomas, are not that same as those involved with SCFA production.

Although SCFAs have been shown to be anti-tumorigenic, most of these
studies have been performed in model systems (16, 17). Additionally,
many of the \emph{in vivo} studies investigate proxies such as fiber
supplementation rather than SCFAs directly (14). Although it is well
known that breakdown products from gut bacteria results in SCFA
production, fiber effects on tumorigenesis may be through other
mechanisms in these \emph{in vivo} model systems. In addition to using
fiber supplementation as a proxy for SCFA treatment, observations on the
benefit of SCFAs in preventing tumorigenesis has been mixed. In previous
case/control studies lower SCFA concentrations between patients with
carcinomas and those without carcinomas was observed (19). Yet, this is
in contrast to multiple randomized-controlled trials that have found no
difference between patients who do and do not get fiber supplementation
to try and prevent tumor recurrence (20, 23). This could be because
fiber has other actions in humans besides providing an energy source for
bacteria to breakdown to make SCFAs. It could also be due to the fact
that SCFA concentrations and responses to fiber can vary quite a bit
between healthy individuals (24). This information taken together with
our observations would suggest that either indviduals who do not respond
to fiber supplementation would need to acquire these bacteria to achieve
a benefit or that SCFAs provide little to no benefit as an
anti-tumorigenic compound in colorectal cancer.

In addition to not having a response to fiber supplementation due to
lack of the required microbes, it is also possible that there are
specific instances of colorectal cancer where SCFAs may be beneficial.
One limitation of current research into the effect of SCFAs in CRC has
been that all tumors are treated as the same type. However, there are
known differences in the types of mutations that occur (25) and treating
all tumors as equal may actually hide any benefit that could be found in
specific subsets of individuals. Similar to the idea of specific
immunotherapy for specific tumors (26), SCFAs may have beneficial
effects for specific types of colorectal tumors. Future research will
need to test if this is a valid hypothesis. Regardless of this
limitation, our results in combination to previous randomized controlled
trials on fiber supplementation suggests that using SCFAs as a general
treatment for colorectal cancer is unlikely to provide a reduction in
tumorigenesis.

One possible technical limitation is that a fecal sample may not be an
ideal type of biospecimen and that the effect SCFAs have on
tumorigenesis is only detected in the colon. However, this is unlikely
to be a major confounder. First, most \emph{in vivo} studies as well as
human studies have used fecal material in their analysis (18, 19).
Second, previous studies that measure SCFA changes after fiber
supplementation use fecal material to track these responses with a great
deal of success (24). Although there are limitations with the current
research on SCFAs and colorectal tumors, technical limitations are less
likely to be cause of this. Additionally, as mentioned earlier, our
observations along with the randomized controlled trials on fiber
supplementation in tumor recurrence (20) provide evidence that these
specific metabolites may not be protective or used as a general
treatment option in colorectal cancer. Yet, taxa that are associated
with SCFA production are consistently higher in indivdiuals without
colon tumors than patients with carcinomas (10, 11, 27).

The potential protection against colorectal cancers may not be from
SCFAs, even though taxa associated with their production are higher in
individuals without tumors versus patients with carcinomas (10, 11, 27).
Indeed our data would support the contention that the taxa are similar
to those associated with SCFA production but that these specific
microbes or OTUs themselves are not associated with SCFAs. In
particular, our results showing that different OTUs from the same
taxonomic classification are in tumor and SCFA Random Forest models
supports this hypothesis. This leads to the possibilty that protection
may be through two other routes. First, there could be a different
pathway or other less extensively studied metabolites that provides the
necessary protection against tumorigenesis. Alternatively, protection
may not occur via a metabolite but instead through niche exclusion of
mouth-associated microbes (e.g. \emph{Fusobacterium},
\emph{Porphyromonas}, \emph{Parvimonas}, \emph{Peptostreptococcus} (6,
12, 13)). The idea of niche exclusion is similar to how the community
protects against \emph{Clostridium difficile} infection (28) with
chronic inflammation replacing the role of antibiotics. Even though we
did not find lower concentrations of SCFAs associated with colorectal
tumors, we think that there are many exciting new avenues to explore
because of these results.

\newpage

\subsection{Conclusions}\label{conclusions}

Our observations found no difference in SCFA concentration, their
utility as a classification tool, or for genes of enzymes involved in
SCFA synthesis between individuals without colon tumors and patients
with either adenoma or carcinoma tumors. Although these results are
different than other reports in the literature, they do align with the
randomized controlled trials that have tested fiber use in preventing
colorectal tumor recurrence. Additionally, these observations suggest
that resident microbes that are not involved in SCFA production may be
the important resident community members involved with preventing
tumorigenesis. By focusing on the alternative mechanisms that are
associated with these non-SCFA producing resident microbes, the
identification of more promising therapeutic options for use in treating
colorectal cancer may be found.

\newpage

\subsection{Materials and Methods}\label{materials-and-methods}

\textbf{Study design and sampling.} The overall protocol has been
described in detail previously (29, 30). In brief, this study used fecal
samples obtained at either a single cross-sectional time point (n=490)
or from before (pre-) and after (post-) treatment of a patient's tumor
(adenoma n =41 and carcinoma n = 26). For patients undergoing treatment
for their tumor the length of time between their initial and follow up
sample ranged from 188 - 546 days. Our use of treatment has been
previously defined as encompassing removal of a tumor with or without
chemotherapy and radiation (29). Diagnosis of tumor was made by
colonoscopic examination and histopathological review of biopsies
obtained (29, 30). The University of Michigan Institutional Review Board
approved the study and informed consent was obtained from all
participants in accordance to the guidelines set out by the Helsinki
Declaration.

\textbf{Measuring specific SCFAs.} Our protocol for the measurement of
acetate, butyrate, and propionate followed a previously published
protocol that used a High-Performance Liquid Chromatography (HPLC)
machine (24). The following changes to this protocol included the use of
frozen fecal samples suspended in 1ml of PBS instead of fecal
suspensions in DNA Genotek OmniGut tubes, and the use of the actual
weight of fecal samples instead of the average weight for SCFA
concentration normalizations. These methodological changes did not
affect the overall median concentrations of these SCFAs between the two
studies (see Table 1 (24) and Figure 1 here).

\textbf{16s rRNA gene sequencing.} The workflow and processing have been
previously described (29, 31, 32). In brief, sequences were quality
filtered and contigs created from the paired end reads. Any sequences
with ambiguous base calls were discarded. Contigs were then checked for
matches to the V4 region of the 16S rRNA gene using the SILVA database
(33). Chimeras were identifed and removed using UCHIME and OTUs
clustered at 97\% similarity (34). The major differences from these
previous reports include: the use of version 1.39.5 of the mothur
software package and clustering Operational Taxonomic Units (OTUs) at
97\% similarity using the OptClust algorithm (35).

\textbf{Generating imputed metagenomes.} The use of PICRUSt version
1.1.2 with the recommended standard operating protocol (21) was used.
Briefly, the mothur shared file and metadata was converted into a biom
formatted table using the biom convert function, the subsequent biom
file was processed with the `normalize\_by\_copy\_number.py' function,
and subsequent imputed metagenomes created using the
`predict\_metagenomes.py' function.

\textbf{Obtaining Operational Protein Families from metagenomes.} A
subset of the cross-sectional group (n=490) containing a total of 85
individuals (normal n=29, adenoma n=28, and carcinoma n=28) was shotgun
sequenced on an Illumina HiSeq using 125 bp paired end reads and a
previously described method (36). Briefly, the sequences were quality
filtered and sequences aligning to the human genome were removed prior
to contig assembly with MEGAHIT (37). Open Reading Frames (ORFs) were
identified using Prodigal (38), counts generated using Diamond (39),
subsequent clustering into Operational Protein Families (OPFs) used
mmseq2 (40), and OPF alignment used the KEGG database (41).

\textbf{Pulling genes involved with SCFA synthesis.} Specific genes
located near the end of the pathways involved in the synthesis of
acetate, butyrate, and propionate were analyzed for any differences
between individuals with normal colons and those with tumors. These
genes were based on pathways from KEGG as well as previous research (41,
42) and a list can be found in the supplemental material {[}Table S1{]}.

\textbf{Random Forest models.} The model was first trained on 80\% of
the data and then tested on the held out 20\% (80/20 split) using the
Random Forest algorithm for classification and regression models via the
caret package (22, 43). This was repeated on 100 different 80/20 splits
of the data to generate a reasonable range for the AUC of the model. The
reported AUCs, unless otherwise specified, are for the test sets. The
classification models were built to group normal versus adenoma, normal
versus carcinoma, and high versus low SCFA concentrations. The
regression models were built to classify the SCFA concentrations of
acetate, butyrate, and propionate regardless of disease status.

\textbf{Statistical analysis workflow.} All analysis was performed using
the statistical language R (44). Generally, a Kruskal-Walis rank sum
test with a Dunn's post-hoc test was used to assess differences between
the groups used. Where appropriate Benjamini-Hochberg was used to
correct for multiple comparisons (45). First, we assessed differences in
SCFA concentrations measured by HPLC between individuals with normal
colons and patients with tumors (adenoma or carcinoma). We then analyzed
whether SCFA concentrations changed in patients with an adenoma or
carcinoma pre- versus post-treatment. Next, the imputed gene counts of
important mediators of SCFA synthesis was tested. Additionally, the
counts generated for OPFs that matched important genes involved with
SCFA creation were analyzed. From here we analyzed the number of
significant positive and negative correlations between OTU relative
abundance and SCFA concentrations in individuals without tumors and
patients with adenomas or carcinomas using Spearman's rho. Next, we
assessed whether OTUs alone or OTUs and SCFAs were better able to
classify individuals with and without tumors using Random Forest models.
Finally, models to classify high or low SCFA concentration based on the
median of each SCFA or the actual concentration using 16S rRNA gene
sequencing data was created using the Random Forest algorithm. For all
Random Forest models, the assessment of the most important variables was
based on the top 10 feautres (OTUs or SCFAs) using the mean decrease in
accuracy.

\newpage

\subsection{Acknowledgements}\label{acknowledgements}

The authors thank the Great Lakes-New England Early Detection Research
Network for providing the fecal samples that were used in this study. We
would also like to thank Kwi Kim and Thomas M Schmidt for their help in
running the short-chain fatty acid analysis on the High-Performance
Liquid Chromatography machine at the University of Michigan. Salary
support for Marc A. Sze came from the Canadian Institute of Health
Research and NIH grant UL1TR002240. Salary support for Patrick D.
Schloss came from NIH grants P30DK034933 and 1R01CA215574.

\newpage

\subsection{References}\label{references}

\hypertarget{refs}{}
\hypertarget{ref-Haggar2009}{}
1. \textbf{Haggar F}, \textbf{Boushey R}. 2009. Colorectal cancer
epidemiology: Incidence, mortality, survival, and risk factors. Clinics
in Colon and Rectal Surgery \textbf{22}:191--197.
doi:\href{https://doi.org/10.1055/s-0029-1242458}{10.1055/s-0029-1242458}.

\hypertarget{ref-Siegel2016}{}
2. \textbf{Siegel RL}, \textbf{Miller KD}, \textbf{Jemal A}. 2016.
Cancer statistics, 2016. CA: A Cancer Journal for Clinicians
\textbf{66}:7--30.
doi:\href{https://doi.org/10.3322/caac.21332}{10.3322/caac.21332}.

\hypertarget{ref-Lichtenstein2000}{}
3. \textbf{Lichtenstein P}, \textbf{Holm NV}, \textbf{Verkasalo PK},
\textbf{Iliadou A}, \textbf{Kaprio J}, \textbf{Koskenvuo M},
\textbf{Pukkala E}, \textbf{Skytthe A}, \textbf{Hemminki K}. 2000.
Environmental and heritable factors in the causation of cancer analyses
of cohorts of twins from sweden, denmark, and finland. New England
Journal of Medicine \textbf{343}:78--85.
doi:\href{https://doi.org/10.1056/nejm200007133430201}{10.1056/nejm200007133430201}.

\hypertarget{ref-FlissIsakov2017}{}
4. \textbf{Fliss-Isakov N}, \textbf{Zelber-Sagi S}, \textbf{Webb M},
\textbf{Halpern Z}, \textbf{Kariv R}. 2017. Smoking habits are strongly
associated with colorectal polyps in a population-based case-control
study. Journal of Clinical Gastroenterology 1.
doi:\href{https://doi.org/10.1097/mcg.0000000000000935}{10.1097/mcg.0000000000000935}.

\hypertarget{ref-Lee2015}{}
5. \textbf{Lee J}, \textbf{Jeon JY}, \textbf{Meyerhardt JA}. 2015. Diet
and lifestyle in survivors of colorectal cancer. Hematology/Oncology
Clinics of North America \textbf{29}:1--27.
doi:\href{https://doi.org/10.1016/j.hoc.2014.09.005}{10.1016/j.hoc.2014.09.005}.

\hypertarget{ref-Kostic2011}{}
6. \textbf{Kostic AD}, \textbf{Gevers D}, \textbf{Pedamallu CS},
\textbf{Michaud M}, \textbf{Duke F}, \textbf{Earl AM}, \textbf{Ojesina
AI}, \textbf{Jung J}, \textbf{Bass AJ}, \textbf{Tabernero J},
\textbf{Baselga J}, \textbf{Liu C}, \textbf{Shivdasani RA},
\textbf{Ogino S}, \textbf{Birren BW}, \textbf{Huttenhower C},
\textbf{Garrett WS}, \textbf{Meyerson M}. 2011. Genomic analysis
identifies association of fusobacterium with colorectal carcinoma.
Genome Research \textbf{22}:292--298.
doi:\href{https://doi.org/10.1101/gr.126573.111}{10.1101/gr.126573.111}.

\hypertarget{ref-Zackular2013}{}
7. \textbf{Zackular JP}, \textbf{Baxter NT}, \textbf{Iverson KD},
\textbf{Sadler WD}, \textbf{Petrosino JF}, \textbf{Chen GY},
\textbf{Schloss PD}. 2013. The gut microbiome modulates colon
tumorigenesis. mBio \textbf{4}:e00692--13--e00692--13.
doi:\href{https://doi.org/10.1128/mbio.00692-13}{10.1128/mbio.00692-13}.

\hypertarget{ref-Baxter2014}{}
8. \textbf{Baxter NT}, \textbf{Zackular JP}, \textbf{Chen GY},
\textbf{Schloss PD}. 2014. Structure of the gut microbiome following
colonization with human feces determines colonic tumor burden.
Microbiome \textbf{2}:20.
doi:\href{https://doi.org/10.1186/2049-2618-2-20}{10.1186/2049-2618-2-20}.

\hypertarget{ref-Zackular2015}{}
9. \textbf{Zackular JP}, \textbf{Baxter NT}, \textbf{Chen GY},
\textbf{Schloss PD}. 2015. Manipulation of the gut microbiota reveals
role in colon tumorigenesis. mSphere \textbf{1}:e00001--15.
doi:\href{https://doi.org/10.1128/msphere.00001-15}{10.1128/msphere.00001-15}.

\hypertarget{ref-Shah2017}{}
10. \textbf{Shah MS}, \textbf{DeSantis TZ}, \textbf{Weinmaier T},
\textbf{McMurdie PJ}, \textbf{Cope JL}, \textbf{Altrichter A},
\textbf{Yamal J-M}, \textbf{Hollister EB}. 2017. Leveraging
sequence-based faecal microbial community survey data to identify a
composite biomarker for colorectal cancer. Gut \textbf{67}:882--891.
doi:\href{https://doi.org/10.1136/gutjnl-2016-313189}{10.1136/gutjnl-2016-313189}.

\hypertarget{ref-meta_analysis_crc_Sze2018}{}
11. \textbf{Sze MA}, \textbf{Schloss PD}. 2018. Leveraging existing 16S
rRNA gene surveys to identify reproducible biomarkers in individuals
with colorectal tumors.
doi:\href{https://doi.org/10.1101/285486}{10.1101/285486}.

\hypertarget{ref-Zeller2014}{}
12. \textbf{Zeller G}, \textbf{Tap J}, \textbf{Voigt AY},
\textbf{Sunagawa S}, \textbf{Kultima JR}, \textbf{Costea PI},
\textbf{Amiot A}, \textbf{Bohm J}, \textbf{Brunetti F},
\textbf{Habermann N}, \textbf{Hercog R}, \textbf{Koch M},
\textbf{Luciani A}, \textbf{Mende DR}, \textbf{Schneider MA},
\textbf{Schrotz-King P}, \textbf{Tournigand C}, \textbf{Nhieu JTV},
\textbf{Yamada T}, \textbf{Zimmermann J}, \textbf{Benes V},
\textbf{Kloor M}, \textbf{Ulrich CM}, \textbf{Knebel Doeberitz M von},
\textbf{Sobhani I}, \textbf{Bork P}. 2014. Potential of fecal microbiota
for early-stage detection of colorectal cancer. Molecular Systems
Biology \textbf{10}:766--766.
doi:\href{https://doi.org/10.15252/msb.20145645}{10.15252/msb.20145645}.

\hypertarget{ref-Baxter2016}{}
13. \textbf{Baxter NT}, \textbf{Ruffin MT}, \textbf{Rogers MAM},
\textbf{Schloss PD}. 2016. Microbiota-based model improves the
sensitivity of fecal immunochemical test for detecting colonic lesions.
Genome Medicine \textbf{8}.
doi:\href{https://doi.org/10.1186/s13073-016-0290-3}{10.1186/s13073-016-0290-3}.

\hypertarget{ref-Holscher2017}{}
14. \textbf{Holscher HD}. 2017. Dietary fiber and prebiotics and the
gastrointestinal microbiota. Gut Microbes \textbf{8}:172--184.
doi:\href{https://doi.org/10.1080/19490976.2017.1290756}{10.1080/19490976.2017.1290756}.

\hypertarget{ref-Louis2016}{}
15. \textbf{Louis P}, \textbf{Flint HJ}. 2016. Formation of propionate
and butyrate by the human colonic microbiota. Environmental Microbiology
\textbf{19}:29--41.
doi:\href{https://doi.org/10.1111/1462-2920.13589}{10.1111/1462-2920.13589}.

\hypertarget{ref-test_OKeefe2016}{}
16. \textbf{O'Keefe SJD}. 2016. Diet, microorganisms and their
metabolites and colon cancer. Nature Reviews Gastroenterology \&
Hepatology \textbf{13}:691--706.
doi:\href{https://doi.org/10.1038/nrgastro.2016.165}{10.1038/nrgastro.2016.165}.

\hypertarget{ref-Encarnao2018}{}
17. \textbf{Encarnação JC}, \textbf{Pires AS}, \textbf{Amaral RA},
\textbf{Gonçalves TJ}, \textbf{Laranjo M}, \textbf{Casalta-Lopes JE},
\textbf{Gonçalves AC}, \textbf{Sarmento-Ribeiro AB}, \textbf{Abrantes
AM}, \textbf{Botelho MF}. 2018. Butyrate, a dietary fiber derivative
that improves irinotecan effect in colon cancer cells. The Journal of
Nutritional Biochemistry \textbf{56}:183--192.
doi:\href{https://doi.org/10.1016/j.jnutbio.2018.02.018}{10.1016/j.jnutbio.2018.02.018}.

\hypertarget{ref-Bishehsari2018}{}
18. \textbf{Bishehsari F}, \textbf{Engen P}, \textbf{Preite N},
\textbf{Tuncil Y}, \textbf{Naqib A}, \textbf{Shaikh M}, \textbf{Rossi
M}, \textbf{Wilber S}, \textbf{Green S}, \textbf{Hamaker B},
\textbf{Khazaie K}, \textbf{Voigt R}, \textbf{Forsyth C},
\textbf{Keshavarzian A}. 2018. Dietary fiber treatment corrects the
composition of gut microbiota, promotes SCFA production, and suppresses
colon carcinogenesis. Genes \textbf{9}:102.
doi:\href{https://doi.org/10.3390/genes9020102}{10.3390/genes9020102}.

\hypertarget{ref-Ohigashi2013}{}
19. \textbf{Ohigashi S}, \textbf{Sudo K}, \textbf{Kobayashi D},
\textbf{Takahashi O}, \textbf{Takahashi T}, \textbf{Asahara T},
\textbf{Nomoto K}, \textbf{Onodera H}. 2013. Changes of the intestinal
microbiota, short chain fatty acids, and fecal pH in patients with
colorectal cancer. Digestive Diseases and Sciences
\textbf{58}:1717--1726.
doi:\href{https://doi.org/10.1007/s10620-012-2526-4}{10.1007/s10620-012-2526-4}.

\hypertarget{ref-Yao2017}{}
20. \textbf{Yao Y}, \textbf{Suo T}, \textbf{Andersson R}, \textbf{Cao
Y}, \textbf{Wang C}, \textbf{Lu J}, \textbf{Chui E}. 2017. Dietary fibre
for the prevention of recurrent colorectal adenomas and carcinomas.
Cochrane Database of Systematic Reviews.
doi:\href{https://doi.org/10.1002/14651858.cd003430.pub2}{10.1002/14651858.cd003430.pub2}.

\hypertarget{ref-Langille2013}{}
21. \textbf{Langille MGI}, \textbf{Zaneveld J}, \textbf{Caporaso JG},
\textbf{McDonald D}, \textbf{Knights D}, \textbf{Reyes JA},
\textbf{Clemente JC}, \textbf{Burkepile DE}, \textbf{Thurber RLV},
\textbf{Knight R}, \textbf{Beiko RG}, \textbf{Huttenhower C}. 2013.
Predictive functional profiling of microbial communities using 16S rRNA
marker gene sequences. Nature Biotechnology \textbf{31}:814--821.
doi:\href{https://doi.org/10.1038/nbt.2676}{10.1038/nbt.2676}.

\hypertarget{ref-randomforest_citation_2002}{}
22. \textbf{Liaw A}, \textbf{Wiener M}. 2002. Classification and
regression by randomForest. R News \textbf{2}:18--22.

\hypertarget{ref-Schatzkin2000}{}
23. \textbf{Schatzkin A}, \textbf{Lanza E}, \textbf{Corle D},
\textbf{Lance P}, \textbf{Iber F}, \textbf{Caan B}, \textbf{Shike M},
\textbf{Weissfeld J}, \textbf{Burt R}, \textbf{Cooper MR},
\textbf{Kikendall JW}, \textbf{Cahill J}, \textbf{Freedman L},
\textbf{Marshall J}, \textbf{Schoen RE}, \textbf{Slattery M}. 2000. Lack
of effect of a low-fat, high-fiber diet on the recurrence of colorectal
adenomas. New England Journal of Medicine \textbf{342}:1149--1155.
doi:\href{https://doi.org/10.1056/nejm200004203421601}{10.1056/nejm200004203421601}.

\hypertarget{ref-scfa_measures_venkataraman2016}{}
24. \textbf{Venkataraman A}, \textbf{Sieber JR}, \textbf{Schmidt AW},
\textbf{Waldron C}, \textbf{Theis KR}, \textbf{Schmidt TM}. 2016.
Variable responses of human microbiomes to dietary supplementation with
resistant starch. Microbiome \textbf{4}.
doi:\href{https://doi.org/10.1186/s40168-016-0178-x}{10.1186/s40168-016-0178-x}.

\hypertarget{ref-Fearon1990}{}
25. \textbf{Fearon ER}, \textbf{Vogelstein B}. 1990. A genetic model for
colorectal tumorigenesis. Cell \textbf{61}:759--767.
doi:\href{https://doi.org/10.1016/0092-8674(90)90186-i}{10.1016/0092-8674(90)90186-i}.

\hypertarget{ref-Thomas2016}{}
26. \textbf{Thomas X}, \textbf{Heiblig M}. 2016. The development of
agents targeting the BCR-ABL tyrosine kinase as philadelphia
chromosome-positive acute lymphoblastic leukemia treatment. Expert
Opinion on Drug Discovery \textbf{11}:1061--1070.
doi:\href{https://doi.org/10.1080/17460441.2016.1227318}{10.1080/17460441.2016.1227318}.

\hypertarget{ref-normalization_Sze2017}{}
27. \textbf{Sze MA}, \textbf{Baxter NT}, \textbf{Ruffin MT},
\textbf{Rogers MAM}, \textbf{Schloss PD}. 2017. Normalization of the
microbiota in patients after treatment for colonic lesions. Microbiome
\textbf{5}.
doi:\href{https://doi.org/10.1186/s40168-017-0366-3}{10.1186/s40168-017-0366-3}.

\hypertarget{ref-Theriot2015}{}
28. \textbf{Theriot CM}, \textbf{Young VB}. 2015. Interactions between
the gastrointestinal microbiome and clostridium difficile. Annual Review
of Microbiology \textbf{69}:445--461.
doi:\href{https://doi.org/10.1146/annurev-micro-091014-104115}{10.1146/annurev-micro-091014-104115}.

\hypertarget{ref-normalization_sze2017}{}
29. \textbf{Sze MA}, \textbf{Baxter NT}, \textbf{Ruffin MT},
\textbf{Rogers MAM}, \textbf{Schloss PD}. 2017. Normalization of the
microbiota in patients after treatment for colonic lesions. Microbiome
\textbf{5}.
doi:\href{https://doi.org/10.1186/s40168-017-0366-3}{10.1186/s40168-017-0366-3}.

\hypertarget{ref-crc_model_baxter2016}{}
30. \textbf{Baxter NT}, \textbf{Ruffin MT}, \textbf{Rogers MAM},
\textbf{Schloss PD}. 2016. Microbiota-based model improves the
sensitivity of fecal immunochemical test for detecting colonic lesions.
Genome Medicine \textbf{8}.
doi:\href{https://doi.org/10.1186/s13073-016-0290-3}{10.1186/s13073-016-0290-3}.

\hypertarget{ref-Schloss2009}{}
31. \textbf{Schloss PD}, \textbf{Westcott SL}, \textbf{Ryabin T},
\textbf{Hall JR}, \textbf{Hartmann M}, \textbf{Hollister EB},
\textbf{Lesniewski RA}, \textbf{Oakley BB}, \textbf{Parks DH},
\textbf{Robinson CJ}, \textbf{Sahl JW}, \textbf{Stres B},
\textbf{Thallinger GG}, \textbf{Horn DJV}, \textbf{Weber CF}. 2009.
Introducing mothur: Open-source, platform-independent,
community-supported software for describing and comparing microbial
communities. Applied and Environmental Microbiology
\textbf{75}:7537--7541.
doi:\href{https://doi.org/10.1128/aem.01541-09}{10.1128/aem.01541-09}.

\hypertarget{ref-Kozich2013}{}
32. \textbf{Kozich JJ}, \textbf{Westcott SL}, \textbf{Baxter NT},
\textbf{Highlander SK}, \textbf{Schloss PD}. 2013. Development of a
dual-index sequencing strategy and curation pipeline for analyzing
amplicon sequence data on the MiSeq illumina sequencing platform.
Applied and Environmental Microbiology \textbf{79}:5112--5120.
doi:\href{https://doi.org/10.1128/aem.01043-13}{10.1128/aem.01043-13}.

\hypertarget{ref-Quast2012}{}
33. \textbf{Quast C}, \textbf{Pruesse E}, \textbf{Yilmaz P},
\textbf{Gerken J}, \textbf{Schweer T}, \textbf{Yarza P}, \textbf{Peplies
J}, \textbf{Glöckner FO}. 2012. The SILVA ribosomal RNA gene database
project: Improved data processing and web-based tools. Nucleic Acids
Research \textbf{41}:D590--D596.
doi:\href{https://doi.org/10.1093/nar/gks1219}{10.1093/nar/gks1219}.

\hypertarget{ref-Edgar2011}{}
34. \textbf{Edgar RC}, \textbf{Haas BJ}, \textbf{Clemente JC},
\textbf{Quince C}, \textbf{Knight R}. 2011. UCHIME improves sensitivity
and speed of chimera detection. Bioinformatics \textbf{27}:2194--2200.
doi:\href{https://doi.org/10.1093/bioinformatics/btr381}{10.1093/bioinformatics/btr381}.

\hypertarget{ref-opticlust_westcott2017}{}
35. \textbf{Westcott SL}, \textbf{Schloss PD}. 2017. OptiClust, an
improved method for assigning amplicon-based sequence data to
operational taxonomic units. mSphere \textbf{2}:e00073--17.
doi:\href{https://doi.org/10.1128/mspheredirect.00073-17}{10.1128/mspheredirect.00073-17}.

\hypertarget{ref-Hannigan2017}{}
36. \textbf{Hannigan GD}, \textbf{Duhaime MB}, \textbf{Ruffin MT},
\textbf{Koumpouras CC}, \textbf{Schloss PD}. 2017. Diagnostic potential
\& the interactive dynamics of the colorectal cancer virome.
doi:\href{https://doi.org/10.1101/152868}{10.1101/152868}.

\hypertarget{ref-Li2015}{}
37. \textbf{Li D}, \textbf{Liu C-M}, \textbf{Luo R}, \textbf{Sadakane
K}, \textbf{Lam T-W}. 2015. MEGAHIT: An ultra-fast single-node solution
for large and complex metagenomics assembly via succinct de bruijn
graph. Bioinformatics \textbf{31}:1674--1676.
doi:\href{https://doi.org/10.1093/bioinformatics/btv033}{10.1093/bioinformatics/btv033}.

\hypertarget{ref-Hyatt2010}{}
38. \textbf{Hyatt D}, \textbf{Chen G-L}, \textbf{LoCascio PF},
\textbf{Land ML}, \textbf{Larimer FW}, \textbf{Hauser LJ}. 2010.
Prodigal: Prokaryotic gene recognition and translation initiation site
identification. BMC Bioinformatics \textbf{11}:119.
doi:\href{https://doi.org/10.1186/1471-2105-11-119}{10.1186/1471-2105-11-119}.

\hypertarget{ref-Buchfink2014}{}
39. \textbf{Buchfink B}, \textbf{Xie C}, \textbf{Huson DH}. 2014. Fast
and sensitive protein alignment using DIAMOND. Nature Methods
\textbf{12}:59--60.
doi:\href{https://doi.org/10.1038/nmeth.3176}{10.1038/nmeth.3176}.

\hypertarget{ref-Steinegger2017}{}
40. \textbf{Steinegger M}, \textbf{Söding J}. 2017. MMseqs2 enables
sensitive protein sequence searching for the analysis of massive data
sets. Nature Biotechnology.
doi:\href{https://doi.org/10.1038/nbt.3988}{10.1038/nbt.3988}.

\hypertarget{ref-Kanehisa2015}{}
41. \textbf{Kanehisa M}, \textbf{Sato Y}, \textbf{Kawashima M},
\textbf{Furumichi M}, \textbf{Tanabe M}. 2015. KEGG as a reference
resource for gene and protein annotation. Nucleic Acids Research
\textbf{44}:D457--D462.
doi:\href{https://doi.org/10.1093/nar/gkv1070}{10.1093/nar/gkv1070}.

\hypertarget{ref-scfa_baxter2014}{}
42. \textbf{Baxter NT}, \textbf{Zackular JP}, \textbf{Chen GY},
\textbf{Schloss PD}. 2014. Structure of the gut microbiome following
colonization with human feces determines colonic tumor burden.
Microbiome \textbf{2}:20.
doi:\href{https://doi.org/10.1186/2049-2618-2-20}{10.1186/2049-2618-2-20}.

\hypertarget{ref-caret_citation}{}
43. \textbf{Jed Wing MKC from}, \textbf{Weston S}, \textbf{Williams A},
\textbf{Keefer C}, \textbf{Engelhardt A}, \textbf{Cooper T},
\textbf{Mayer Z}, \textbf{Kenkel B}, \textbf{R Core Team},
\textbf{Benesty M}, \textbf{Lescarbeau R}, \textbf{Ziem A},
\textbf{Scrucca L}, \textbf{Tang Y}, \textbf{Candan C}, \textbf{Hunt.
T}. 2017. Caret: Classification and regression training.

\hypertarget{ref-r_citation_2017}{}
44. \textbf{R Core Team}. 2017. R: A language and environment for
statistical computing. R Foundation for Statistical Computing, Vienna,
Austria.

\hypertarget{ref-benjamini_controlling_1995}{}
45. \textbf{Benjamini Y}, \textbf{Hochberg Y}. 1995. Controlling the
false discovery rate: A practical and powerful approach to multiple
testing. Journal of the Royal Statistical Society Series B
(Methodological) \textbf{57}:289--300.

\newpage

\textbf{Figure 1. No change in SCFA measurements was observed between
normal, adenoma, and carcinoma individuals using HPLC.} Acetate
concentrations in fecal samples of individuals without colon tumors,
adenomas, and carcinomas (A). Butyrate concentrations in fecal samples
of individuals without colon tumors, adenomas, and carcinomas (B).
Propionate concentrations in fecal samples of individuals without colon
tumors, adenomas, and carcinomas (C). The black lines indicate the
median SCFA concentration. Acetate concentrations in fecal samples
before and after treatment for adenoma (yellow) and carcinoma (red) (D).
Butyrate concentrations in fecal samples before and after treatment for
adenoma (yellow) and carcinoma (red) (E). Propionate concentrations in
fecal samples before and after treatment for adenoma (yellow) and
carcinoma (red) (F). The black dots and lines represent the median
change in SCFA concentration.

\textbf{Figure 2. No change in butyrate producing genes identified
between normal, adenoma, and carcinoma individuals.} Imputed gene
relative abundance of important butyrate pathway genes using PICRUSt
(A). Counts per million (corrected for size and number of contigs in an
OPF) for the Butyrate Kinase gene (B). The other butyrate pathway genes
from the PICRUSt analysis did not align to any of the OPFs in the
metagenome analysis.

\textbf{Figure 3. Patients with adenomas had the higest number of
significant negative correlations between OTU relative abundance and
SCFA concentration.} Colors denote the family or lowest taxonomic ID
that an OTU classified to. Fewer significant positive correlations were
observed overall. Additionally, the differences in the number of
significant positive correlations between patients with adenomas versus
individuals without tumors (normal) and patients with carcinomas was not
as pronounced as the number of significant negative correlations.

\textbf{Figure 4. SCFA concentrations do not improve OTU-based Random
Forest models.} The area under the curve of 100 different 80/20 split
OTU-based normal versus adenoma 10-fold CV models with and without SCFAs
(A). The top 10 most important OTUs or SCFAs in the SCFA and OTU adenoma
model (B). The top 10 most important OTUs in the OTU adenoma model (C).
The area under the curve of 100 different 80/20 OTU-based normal versus
carcinoma 10-fold CV models with and without SCFAs (D). The top 10 most
important OTUs or SCFAs in the SCFA and OTU carcinoma model (E). The top
10 most important OTUs in the OTU carcinoma model (F). For (A) and (D)
the black line represents the median AUC. The dotted line highlights an
AUC of 0.5.

\textbf{Figure 5. OTU-based regression Random Forest models of SCFA
concentrations.} The train and test correlation between actual and
predicted values from 100 different 80/20 split OTU-based models with
10-fold CV using regression Random Forest (A). The model accuracy of
predicted SCFA concentrations differed between individuals without
tumors, patients with adenomas, and patients with carcinomas. Generally,
patients with carcinomas had predicted concentrations closest to their
actual measured concentration (B). The top 10 OTUs based on mean
decrease in accuracy (MDA) for each SCFA model, colored by their lowest
taxonomic identification (C).

\newpage

\textbf{Figure S1. Patients with adenomas had the higest number of
significant differences in OTU relative abundance between high/low SCFA
groups.} Colors denote the family or lowest taxonomic ID that an OTU
classified to. Fewer significant OTUs were observed in individuals
without tumors (normal) and patients with carcinomas versus patients
with adenomas.

\textbf{Figure S2. OTU-based classification Random Forest models of
high/low SCFA groups based on overall SCFA median concentration.} The
train and test results of 100 different 80/20 split OTU-based models
with 10-fold CV based on higher or lower than the median SCFA
concentration using classification Random Forest (A). The model accuracy
of predicted high/low SCFA groups differed between individuals without
tumors, patients with adenomas, and patients with carcinomas. Patients
with adenomas consistently had the best classification accuracy (B). The
top 10 OTUs based on mean decrease in accuracy (MDA) for each SCFA
model, colored by their lowest taxonomic identification (C).


\end{document}
