---
title: "Revisiting the Relationship between Short-Chain Fatty Acids, the Microbiota, and Colorectal Tumors"
bibliography: references.bib
output:
  pdf_document:
    keep_tex: true
    includes:
      in_header: header.tex
csl: mbio.csl
fontsize: 11pt
geometry: margin=1.0in
---

```{r knitr_settings, tidy=TRUE, eval=TRUE, echo=FALSE, cache=FALSE}
options(tidyverse.quiet = TRUE)

library(tidyverse)
library(knitr)

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("message" = FALSE)
opts_chunk$set("cache" = FALSE)

inline_hook <- function(x){
  print(x)
  if(is.list(x)){
    x <- unlist(x)
  }
  if(is.numeric(x)){
    if(abs(x - round(x)) < .Machine$double.eps^0.5){
      paste(format(x,big.mark=',', digits=0, scientific=FALSE))
    } else {
      paste(format(x,big.mark=',', digits=2, nsmall=2, scientific=FALSE))
    }
  } else {
      paste(x)
  }
}
knitr::knit_hooks$set(inline=inline_hook)
```


\vspace{35mm}

Fecal short chain fatty acids are not predictive of colorectal cancer status and cannot be predicted based on bacterial community structure

\vspace{35mm}


Marc A. Sze${^1}$, Begüm Topçuoğlu${^1}$, Nicholas A. Lesniak${^1}$, Mack T. Ruffin IV${^2}$, Patrick D. Schloss${^1}$${^\dagger}$

\vspace{40mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI 48109

$2$ Department of Family Medicine and Community Medicine, Penn State Hershey Medical Center, Hershey, PA


\newpage
\linenumbers


## Abstract (250 words)

Something clever


## Importance (150 words)

Something clever

\newpage

Colorectal cancer is the third leading cancer-related cause of death within the United States [@Siegel2016]. Less than 10% of cases can be attributed to genetic risk factors [@Fearon1990]. This leaves a significant role for environmental,  behavioral, and other factors such as smoking and diet [@FlissIsakov2017; @Lee2015]. Colorectal cancer is thought to be initiated by a series of mutations that accumulate as mutated cells proliferate leading to adenomatous lesions, which are succeeded by carcinomas [@Fearon1990]. Throughout this progression, there are ample opportunities for bacterial populations to create mutations, induce inflammation, and accelerate tumorigenesis. Numerous studies in murine models have supported this model [@Zackular2013; @DeStefanoShields2016; @Tomkovich2017]. Additional cross sectional studies in humans have identified microbiome-based biomarkers of disease [@Sze2018]. These studies suggest that in some cases, it is the loss of bacterial populations that produce short-chain fatty acids (SCFAs) that results in increased inflammation and tumorigenesis.

SCFAs have have anti-inflammatory and anti-proliferative activities [@OKeefe2016]. Manipulation of SCFAs in mouse models of colorectal cancer by direct supplementation or feeding of fiber caused an overall reduction in tumor burden [@Bishehsari2018]. These results suggest that supplementation with substrates that bacteria can ferment to produce SCFAs may confer beneficial effects against colorectal cancer. Regardless, there is a lack of evidence that increasing SCFA concentrations can protect against colorectal cancer in humans. Based on similar observations, many microbiome studies use the concentrations of SCFAs and the presence of 16S rRNA gene sequences from organisms and the genes involved in producing them as a biomarker of a healthy microbiota [@Sanna2019; @Meisel2016]. Case-control studies that have investigated SCFA concentrations in colorectal cancer found that patients with carcinomas had lower concentrations of SCFAs versus patients with adenomas or individuals without colon tumors [@Ohigashi2013]. Although this would argue that increasing SCFA concentrations could be protective against tumorigenesis, in randomized controlled trials fiber supplementation has been inconsistently associated with protection against tumor formation and recurrence [@Yao2017; @Gianfredi2018]. These findings temper enthusiasm for treatments that aim to use SCFAs as biomarkers or protection against tumorigenesis.


```{r, concentration}
n <- read_csv('../data/metadata/cross_section.csv') %>% count(dx)
n_normal <- n %>% filter(dx == "normal") %>% pull(n)
n_adenoma <- n %>% filter(dx == "adenoma") %>% pull(n)
n_cancer <- n %>% filter(dx == "cancer") %>% pull(n)

cross_test <- read_tsv('../data/process/scfa_cross_section_stats.tsv')
iso_p_cross <- cross_test %>% filter(p.value < 0.05)
stopifnot(nrow(iso_p_cross) == 1)

n_follow <- read_csv('../data/metadata/follow_up.csv') %>% count(dx)
n_follow_adenoma <- n_follow %>% filter(dx == "adenoma") %>% pull(n)
n_follow_cancer <- n_follow %>% filter(dx == "cancer") %>% pull(n)

follow_up_time <- read_csv('../data/metadata/follow_up.csv') %>%
 	summarize(q1=quantile(time, prob=0.25), median=median(time), q3=quantile(time, prob=0.75))

follow_test <- read_tsv('../data/process/scfa_pre_post_stats.tsv')
iso_p_follow <- follow_test %>% filter(p.value < 0.05)
stopifnot(nrow(iso_p_follow) == 1)

pooled_test <- min(c(cross_test %>% filter(scfa=="pooled") %>% pull(p.value), follow_test %>% filter(scfa=="pooled") %>% pull(p.value)))

```

**SCFA concentrations do not meaningfully vary with diagnosis or treatment.** To quantify the associations between colorectal cancer, the microbiome, and SCFAs, we quantified the concentration of acetate, propionate, isobutyrate, and butyrate in feces of previously characterized individuals with normal colons (N=`r n_normal`) and those with colonic adenomas (N=`r n_adenoma`) or carcinomas (N=`r n_cancer`) [@Baxter2016]. The only SCFA that had a significantly different concentration across the diagnoses was isobutyrate (P=`r iso_p_cross$p.value`; Figure 1A). Interestingly, the median concentration of isobutyrate was `r iso_p_cross$normal` mmol/kg in people with normal colons and it was `r iso_p_cross$adenoma` and `r iso_p_cross$cancer` mmol/kg in people with colonic adenomas or carcinomas, respectively. The difference in isobutyrate concentration between people with adenomas and carcinomas was significantly different (P=`r iso_p_cross["cancer-adenoma"]`); however, the differences between people with normal colons and those with adenomas or carcinomas was not significant (P=`r iso_p_cross["normal-adenoma"]` and P=`r iso_p_cross["normal-cancer"]`). Among the subjects with adenomas and carcinomas, a subset (N~adenoma~=`r n_follow_adenoma`, N~carcinoma~=`r n_follow_cancer`) were treated and sampled a year later [@Sze2017]. The only SCFA that changed following treatment was isobutyrate, which decreased by `r iso_p_follow$estimate` mmol/kg (P=`r iso_p_follow$p.value`; Figure 1B). For both the pre-treatment cross-sectional data and the pre/post treatment data, we pooled the SCFA concentrations on a per molecule of carbon basis and again failed to see any significant differences (P>`r pooled_test`). The low concentration of isobutyrate relative to the other SCFAs, inconsistent concentrations, and unexpected decrease in concentration with treatment makes it difficult to ascribe much biological relevance to this observation.

```{r, classification}
random <- read_tsv("../data/rf/classification_SCFA_to_random.tsv")
stopifnot(random$p.value < 0.001)

w_wo_compare <- read_tsv("../data/rf/classification_w_wo_SCFA.tsv")

min_p_compare_tax <- w_wo_compare %>%
	filter(class=="adenoma" | class == "cancer") %>%
	filter(microbiome =="otu" | microbiome == "genus") %>%
  select(p.value) %>%
	min()

min_p_compare_metag <- w_wo_compare %>%
	filter(class=="adenoma" | class == "cancer") %>%
	filter(microbiome =="opf" | microbiome == "kegg") %>%
	select(p.value) %>%
	min()

n_s <- read_tsv("../data/metagenome/metag.sample.counts")
```

**Combining SCFA and microbiome data does not improve the ability to diagnose individual as having adenomas or carcinomas.** We previously found that binning 16S rRNA gene sequence data into operational taxonomic units based on 97% similarity or into genera enabled us to classify individuals as having adenomas or carcinomas using Random Forest machine learning models [@Baxter2016; @Sze2018]. We repeated that analysis but added the concentration of the individual or pooled SCFAs as possible features to train a model. Models trained using SCFAs to classify individuals as having adenomas or carcinomas rather than normal colons had median areas under the receiver operator characteristic curve (AUROC) that were significantly greater than 0.5 (P~adenoma~<0.001 and P~carcinoma~<0.001); however, the AUROC values to detect the presence of adenomas or carcinomas were only `r random %>% filter(class=="adenoma") %>% pull(median)` and `r random %>% filter(class=="cancer") %>% pull(median)`, respectively (Figure 2A). When we trained the models with the SCFAs concentrations and operational taxonomic unit (OTU) or genus-level relative abundances the AUROC values were not significantly different from the models trained without the SCFA concentrations (P>`r min_p_compare_tax`; Figure 2A). We also trained models using a smaller dataset that generated shotgun metagenomic sequencing data from a subset of our cohort (N~normal~=`r n_s %>% filter(dx == "normal") %>% pull(n)`, N~adenoma~=`r n_s %>% filter(dx == "adenoma") %>% pull(n)`, and N~cancer~=`r n_s %>% filter(dx == "cancer") %>% pull(n)`) [@Hannigan2017]. We binned genes extracted from the assembled metagenomes into operational protein families (OPFs) or KEGG categories. Again, the performance of the models trained with the meteagenomic data did not improve when the SCFA concentrations were added as possible features when training the model (P>`r min_p_compare_metag`; Figure 2A). These data demonstrate that knowledge of the SCFA profile from a patient's fecal sample does not improve the ability to diagnose a colonic lesion.

```{r, regression}
```

**Knowledge of microbial community structure does not predict SCFA concentrations.** Regardless of a person's diagnosis, we next asked whether the fecal community structure was predictive of fecal SCFA concentrations. We trained Random Forest regression models using 16S rRNA gene sequence data binned into OTUs and genera to predict the concentration of the individual and pooled SCFAs. Regardless of the binning method or SCFA, the largest amount of variation that the model could explain was **0.XXXX** (Figure 2B). Next, we trained Random Forest regression models using metagenomic sequence data binned into OPFs and KEGG categories to predict the concentration of the individual and pooled SCFAs. Similar to the analysis using 16S rRNA gene sequence data, the metagenomic data was not predictive of SCFA concentration; the largest amount of variation that the models could explain was **0.XXXX** (Figure 2B). Because of the limited number of samples that we were able to generate metagenomic sequence data from, we used our 16S rRNA gene sequence data to impute metagenomes that were binned into metabolic pathways, enzyme commission numbers, or KEGG categories using PICRUSt. Again, SCFA concentrations could not be predicted based on the imputed metagenomic data; the largest amount of variation that the models could explain was **0.XXXX** (Figure 2B). The inability to model SCFA concentrations from microbiome data indicates that the knowledge of the abundance of organisms and genes is insufficient to predict SCFA concentrations.


**Conclusion.** Our data indicate that fecal SCFA concentrations are not associated with the presence of adenomas or carcinomas and that they provide weak predictive power to improve the ability to diagnose someone with one of these lesions. Furthermore, knowledge of the taxonomic and genetic structure of gut microbiota was not predictive of individual or pooled SCFA concentrations. These results complement existing literature that suggest that fiber consumption and the production of SCFAs are unable to prevent the risk of developing colonic tumors. It is important to note that our analysis concerned fecal SCFA concentrations and microbiome characterization and that observations along the mucosa near the site of lesions may provide a stronger association. Regardless, given the growing literature in this area, it is unlikely that SCFAs are the primary mechanism that limits tumorigenesis. This may be a cautionary result to temper enthusiasm for SCFAs as a biomarker of gut health more generally. Going forward it is critical to develop additional hypotheses for how the microbiome and host interact to drive tumorigenesis to better understand the disease process and identify biomarkers that will allow early detection of tumors.



\newpage

## Materials and Methods

**Study design and sampling.** The overall study design and the resulting sequence data have been previously described [@Sze2017; @Baxter2016]. In brief, fecal samples were obtained from `r n_normal` individuals with normal colons, `r n_adenoma` individuals with colonic adenomas, and `r n_cancer` individuals with carcinomas. Of the individuals diagnosed as having adenomas or carcinomas, a subset (N~adenoma~=`r n_follow_adenoma` and N~carcinoma~=`r n_follow_cancer`) were sampled after treatment of the lesion (median=`r follow_up_time$median` days between sampling, IQR=`r follow_up_time$q1` to `r follow_up_time$q3` days). Tumor diagnosis was made by colonoscopic examination and histopathological review of the biopsies [@Baxter2016]. The University of Michigan Institutional Review Board approved the studies that generated the samples and informed consent was obtained from all participants in accordance to the guidelines set out by the Helsinki Declaration.

**Measuring specific SCFAs.** The measurement of acetate, propionate, isobutyrate, and butyrate used a previously published protocol that used High-Performance Liquid Chromatography (HPLC) [@Venkataraman2016]. Two changes were made to the protocol. First, instead of using fecal samples suspended in DNA Genotek OmniGut tubes, we suspended frozen fecal samples in 1 mL of PBS. Second, instead of using the average weight of fecal sample aliquots to normalize SCFA concentrations, we used the actual weight of the fecal samples. These methodological changes did not affect the range of concentrations of these SCFAs between the two studies.

**16S rRNA gene sequence data analysis.** Sequence data from Baxter et al. [@Baxter2016] and Sze et al. [@Sze2017] were obtained from the Sequence Read Archive (studies SRP062005 and SRP096978) and reprocessed using using mothur v.1.42 [@Schloss2009]. The original studies generated sequence data from V4 region of the 16S rRNA gene using paired 250 nt reads on an Illumina MiSeq sequencer. The resulting sequence data were assembled into contigs, screened to remove low quality contigs and chimeras. The curated sequences were then clustered into OTUs at a 97% similarity threshold and assigned to the closest possible genus with an 80% confidence threshold trained on the reference collection from the Ribosomal Database Project (v.16). We used PICRUSt (v.2.1.0-b) with the recommended standard operating protocol to generate imputed metagenomes based on the expected metabolic pathways, KEGG categories, and enzyme commission numbers [@Langille2013].

**Metagenomic DNA sequence analysis.** A subset of the samples from the samples described by Baxter et al. [@Baxter2016] were used to generate metagenomic sequence data (N~normal~=`r n_s %>% filter(dx == "normal") %>% pull(n)`, N~adenoma~=`r n_s %>% filter(dx == "adenoma") %>% pull(n)`, and N~cancer~=`r n_s %>% filter(dx == "cancer") %>% pull(n)`). These data were generated by Hannigan et al. [@Hannigan2017] and deposited into the Sequence Read Archive (study SRP108915). Fecal DNA was subjected to shotgun sequencing on an Illumina HiSeq using 125 bp paired end reads. The archived sequences were already quality filtered and aligned to the human genome to remove contaminating sequence data. We downloaded the sequences and assembled them into contigs using MEGAHIT [@Li2015], which were used to identify open reading frames (ORFs) using Prodigal [@Hyatt2010]. We determined the abundance of each ORF by mapping the raw reads back to the ORFs using Diamond [@Buchfink2014]. We clustered the ORFs into operational protein families (OPFs) in which the clustered ORFs were more than 40% identical to each other using mmseq2 [@Steinegger2017]. We also used mmseq2 to map the ORFs to the KEGG database and clustered the ORFs according to which category the ORFs mapped.

**Random Forest models.**















**Statistical analysis workflow.** Data summaries, statistical analysis, and data visualizations were performed using R (v.3.5.1) with the tidyverse package (v.1.2.1). To assess differences in SCFA concentrations between individuals normal colons and those with adenomas or carcinomas, we used the Kruskal-Wallis rank sum test. If a test had a P-value below 0.05, we then applied a pairwise Wilcoxon rank sum test with a Benjamini-Hochberg correction for multiple comparisons. To assess differences in SCFA concentrations between individuals samples before and after treatment we used paired Wilcoxon rank sum tests to test for significance. To compare the median AUCROC for the held out data for the model generated using only the SCFAs, we compared the distribution of the data to the expected median of 0.5 using the Wilcoxon rank sum test to test whether the model performed better than would be achieved by randomly assigning the data to each diagnosis. When we compared the Random Forest models generated without and with SCFA data included, we used Wilcoxon rank sum tests to determine whether the models with the SCFA data included did better.


**Code availability.** The code for all sequence curation and analysis steps including an Rmarkdown version of this manuscript is available at https://github.com/SchlossLab/Sze_SCFACRC_XXXX_2019/.


\newpage

## Acknowledgements

The authors thank the Great Lakes-New England Early Detection Research Network for providing the fecal samples that were used in this study. We would thank the University of Michigan Center for Microbial Systems for enabling our short-chain fatty acid analysis. Support for MAS came from the Canadian Institute of Health Research and the National Institutes of Health (UL1TR002240). Support for PDS came from the National Institutes of Health (P30DK034933 and R01CA215574).   

\newpage

## References

<div id="refs"></div>

\newpage

\includegraphics{../results/figures/scfa_abundance.pdf}

**Figure 1. SCFA concentrations did not vary meaningfully with diagnosis of colonic lesions or with treatment for adenomas or carcinomas.** (A) We measured the concentration of fecal SCFAs from individuals with normal colons (N=`r n_normal`) or those with adenoma (N=`r n_adenoma`) or carcinomas (N=`r n_cancer`) was for isobutyrate. (B) A subset of individuals diagnosed with adenomas (N=`r n_follow_adenoma`) or carcinomas (N=`r n_follow_cancer`) who underwent treatment were resampled a year after the initial sampling; one extreme propionate value (124.4 mmol/kg) was included in the adenoma analysis but censored from the visualization for clarity.

\newpage

**Figure 2. SCFA concentrations do not improve models for diagnosing the presence of adenomas, carcinomas, or all lesions. 16S rRNA gene and metagenomic sequence data do not predict SCFAs concentrations.**


\newpage


**Figure S1. Comparison of training and testing results.**
