\documentclass[11pt,]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage[margin=1.0in]{geometry}
\usepackage{hyperref}
\hypersetup{unicode=true,
            pdftitle={Revisiting the Relationship between Short-Chain Fatty Acids, the Microbiota, and Colorectal Tumors},
            pdfborder={0 0 0},
            breaklinks=true}
\urlstyle{same}  % don't use monospace font for urls
\usepackage{graphicx,grffile}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
\IfFileExists{parskip.sty}{%
\usepackage{parskip}
}{% else
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{0}
% Redefines (sub)paragraphs to behave more like sections
\ifx\paragraph\undefined\else
\let\oldparagraph\paragraph
\renewcommand{\paragraph}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
\let\oldsubparagraph\subparagraph
\renewcommand{\subparagraph}[1]{\oldsubparagraph{#1}\mbox{}}
\fi

%%% Use protect on footnotes to avoid problems with footnotes in titles
\let\rmarkdownfootnote\footnote%
\def\footnote{\protect\rmarkdownfootnote}

%%% Change title format to be more compact
\usepackage{titling}

% Create subtitle command for use in maketitle
\newcommand{\subtitle}[1]{
  \posttitle{
    \begin{center}\large#1\end{center}
    }
}

\setlength{\droptitle}{-2em}
  \title{Revisiting the Relationship between Short-Chain Fatty Acids, the
Microbiota, and Colorectal Tumors}
  \pretitle{\vspace{\droptitle}\centering\huge}
  \posttitle{\par}
  \author{}
  \preauthor{}\postauthor{}
  \date{}
  \predate{}\postdate{}

\usepackage{helvet} % Helvetica font
\renewcommand*\familydefault{\sfdefault} % Use the sans serif version of the font
\usepackage[T1]{fontenc}

\usepackage[none]{hyphenat}

\usepackage{setspace}
\doublespacing
\setlength{\parskip}{1em}

\usepackage{lineno}

\usepackage{pdfpages}

\begin{document}
\maketitle

\vspace{35mm}

Running title: SCFAs and colorectal tumors

\vspace{35mm}

Marc A Sze\({^1}\), Nicholas A Lesniak\({^1}\), Mack T Ruffin
IV\({^2}\), Patrick D. Schloss\({^1}\)\({^\dagger}\)

\vspace{40mm}

\(\dagger\) To whom correspondence should be addressed:
\href{mailto:pschloss@umich.edu}{\nolinkurl{pschloss@umich.edu}}

\(1\) Department of Microbiology and Immunology, University of Michigan,
Ann Arbor, MI 48109

\(2\) Department of Family Medicine and Community Medicine, Penn State
Hershey Medical Center, Hershey, PA

\newpage

\linenumbers

\subsection{Abstract}\label{abstract}

\textbf{Background.} Colorectal cancer (CRC) is a growing health concern
with the majority of the risk for developing disease being due to
environmental factors. The microbiota is one of these environmental
factors with certain bacterial community members being associated with
CRC, while other taxa are associated to colons without tumors. Some of
the taxa associated to colons without tumors can use fiber to produce
short-chain fatty acids (SCFAs) that can inhibit tumor growth in model
systems. However, the data supporting the importance of SCFAs in human
CRC is less certain. Here, we test the hypothesis that SCFA
concentrations are different in individuals with colorectal tumors.

\textbf{Methods.} We analyzed a cross-sectional (n=490) and longitudinal
pre- and post-treatment (n=67) group for their concentrations of
acetate, butyrate, and propionate. Analysis also included tumor
classification models using Random Forest, imputed gene relative
abundance with PICRUSt, and metagenomic sequencing on a subset (n=85) of
the total cross-sectional group.

\textbf{Results.} No difference in SCFA concentrations were found
between individuals without tumors and patients with adenomas or
carcinomas (P-value \textgreater{} 0.15). There was no difference in
classification models with or without SCFAs in their ability to predict
patients with adenomas or carcinomas versus individuals without tumors
(P-value \textgreater{} 0.05). Using metagenomic sequencing, there was
also no difference in genes involved with SCFA synthesis between
individuals without tumors and patients with adenomas or carcinomas
(P-value \textgreater{} 0.70).

\textbf{Conclusions.} Although our data does not support the hypothesis
that SCFAs are different in individuals that have colorectal tumors,
there may be context specific scenarios where SCFAs may still be
beneficial for treatment of CRC. Alternatively, there may be other
mechanisms that have not been thoroughly investigated that are more
important to the development of human CRC.

\newpage

\subsection{Introduction}\label{introduction}

Colorectal cancer (CRC) is currently the third leading cancer-related
cause of death within the US (1, 2). Although there is a genetic
component to the disease, the environment has been attributed as an
important and large risk factor for CRC (3). This has lead to the
investigation of how the microbiota may exacerbate or cause tumorigensis
(4--6) and whether the bacterial community is altered (7, 8). Multiple
reports in case/control studies on the microbiota and CRC have
identified taxa commonly associated with the normal colons to be
decreased in patients with carcinoma tumors (9--11). Many of these taxa
actively produce short-chain fatty acids (SCFAs) from fiber that are a
part of our general diet (12). The most extensively studied of these
SCFAs are acetate, butyrate, and propionate (13). Overall, CRC is a
growing health problem and the microbiota through SCFAs have shown
promise as a potential treatment option.

Specific SCFAs, like butyrate, have shown positive results for CRC
treatment within model systems (14). Butyrate has been shown to inhibit
cancer cell growth in \emph{in vitro} systems (15). Additionally,
supplementation with food sources that bacteria use to create these
SCFAs may also be able to confer beneficial effects. For example, fiber
supplementation in mouse models of CRC caused an overall reduction in
tumor burden while also increasing SCFA concentrations (16). Although
these model systems provide important preliminary evidence towards the
ability of SCFAs to reduce and treat tumors, the studies reporting
benefit in humans has been less convincing.

There is a lack of evidence on the benefit of increasing SCFA
concentrations to protect against CRC in human populations. The initial
case/control studies that investigated SCFA concentrations in CRC found
that patients with carcinomas had lower concentrations of acetate,
butyrate, and propionate versus either patients with adenomas or
individuals with normal colons (17). Although this would argue that
increasing SCFA concentrations could be protective against
tumorigenesis, fiber supplementation in randomized controlled trials
have consistently failed to protect against tumor recurrence (18). These
findings would argue against the utility of treatments that aim to use
SCFAs to reduce or protect against tumorigenesis. Given the lack of
clear evidence in human studies of the benefit of SCFAs in CRC, there is
a need for more investigation into this area.

Our current study will fill some of the current gaps in the literature
that relate to the study of SCFAs and CRC in human populations.
Specifically, it will attempt to replicate previous case/control
findings on SCFA concentrations in individuals with and without tumors.
Additionally, by increasing the adenoma tumor cohort it we will identify
if the observation of a continuous gradual reduction in SCFA
concentrations occurs as tumor severity increases. It also will build
upon these observations and assess the utility of using SCFAs and
Operational Taxonomic Units (OTUs) as a potential risk stratification
tool of colorectal tumors (adenoma or carcinoma). This study will
provide important information on the replicability of previous findings
in humans by extensively studying how SCFAs could be associated with
colorectal tumors.

To accomplish this task we will directly measure the concentration of
acetate, butyrate, and propionate within fecal samples for two different
groups. The first group had a sample obtained at a single cross
sectional point in time while the second group had samples obtained
before (pre-) and after (post-)treatment for colorectal tumors.
Additionally, we also assessed whether adding SCFA concentrations to OTU
data improves the classification of adenoma or carcinoma using the
Random Forest algorithm (19), used PICRUSt (20) and metagenomic
sequencing to assess the genes involved in SCFA synthesis, and analyzed
how well 16S rRNA gene sequencing can predict SCFA concentrations. This
investigation provides additional information as to whether the taxa
that are decreased in carcinoma use SCFAs to help protect against colon
tumorigenesis.

\newpage

\subsection{Results}\label{results}

\textbf{Decreased SCFA concentrations are not associated with tumors.}
We used high-performance liquid chromatography (HPLC) to measure
acetate, butyrate, and propionate concentrations of frozen fecal samples
from 490 individuals at a cross-sectional point in time. There was no
difference between individuals with normal colons (n=172) and patients
with either an adenoma (n=198) or carcinoma (n=120) for any of the SCFAs
measured after multiple comparison correction (P-value \textgreater{}
0.15) {[}Figure 1A - 1C{]}. We next measured the concentration of SCFAs
in 67 patients with an adenoma (n=41) or carcinoma (n=26) in which we
had pre- and post-treatment fecal samples. Although there was a general
trend for increasing acetate, butyrate, and propionate concentrations
after treatment for tumors, there was no significant difference pre- and
post-treatment for patients with adenoma (P-value \textgreater{} 0.20)
or carcinoma (P-value \textgreater{} 0.80) {[}Figure 1D - 1F{]}. Even
though there was no change in SCFA concentrations between individuals
with normal colons and those with tumors, this information could still
be important to help classify disease.

\textbf{Random Forest models with SCFA concentrations do not classify
tumors better.} Using the Random Forest algorithm we built models with
OTUs only or OTUs and SCFA concentrations to classify normal versus
adenoma and normal versus carcinoma fecal samples. For adenoma and
carcinoma models, there was no difference between the median AUC of
models with or without SCFA concentrations (P-value \textgreater{} 0.05)
{[}Figure 2{]}. Although including SCFA concentrations did not add extra
information for classification of colorectal tumors using Random Forest
models, it is possible that the genes for enzymes involved in SCFA
synthesis may vary based on the type of colorectal tumor; where a
smaller number of microbes are responsible for the observed SCFA
concentrations in patients with carcinoma.

\textbf{Changes in genes for enzymes involved in SCFA synthesis are not
associated with tumors.} Using a list of specific genes that are
important for the synthesis of SCFAs {[}Table S1{]}, we looked for
differences in gene abundance based on individuals having normal colons,
adenomas, or carcinomas. First, using imputed gene relative abundance
based on 16S rRNA gene sequencing we found no difference in any of the
genes involved with acetate, butyrate, or propionate synthesis (P-value
\textgreater{} 0.90) {[}Table S2{]}. This similarity between groups is
highlighted by visualizing genes important in butyrate synthesis
{[}Figure 3A{]}. Using a Wilcoxon rank-sum test, there also was no
difference in imputed gene relative abundance between pre- and
post-treatment samples for any genes involved with SCFA synthesis
(P-value \textgreater{} 0.70) {[}Table S3{]}. Next, we took a subset of
these 490 fecal samples (n=85) and used metagenomic sequencing to
confirm these results. Like the imputed gene results, metagenomic
analysis found that there was no difference in any of the genes involved
in SCFA synthesis between individuals with normal colons (n=29) and
patients with adenoma (n=28) or carcinoma (n=28) (P-value \textgreater{}
0.70) {[}Table S4{]}. This lack of difference is highlighted when we
visualize the results for butyrate kinase {[}Figure 3B{]}. These
observations provide evidence that the gene content does not change due
to colorectal tumors. Although these results suggest that SCFA
concentrations do not change due to colorectal tumor, there may be some
errors present within our results that may account for the lack of
associations.

\textbf{Expected taxa are associated with higher SCFA concentrations
regardless of tumor status.} Using OTU data we built Random Forest
models to classify higher than median and lower than median SCFA
concentrations. Overall, OTU data had a reasonable ability to classify
high and low SCFA concentrations {[}Figure S1A{]}. However, these models
tended to be over fit, suggesting that rarer taxa may be important for
this classification {[}Figure S1A{]}. The most important OTUs to these
models (assessed with mean decrease in accuracy (MDA)) were to taxa that
are normally associated with SCFA production {[}Figure S1B{]}. These
results highlight that there are no immediately apparent errors in our
data because SCFA concentrations are associated with taxa known to
produce acetate, butyrate, and propionate. Additionally, OTUs associated
with these taxa are the most important to models that can classify high
and low SCFA concentrations. Overall, our results are robust and do not
support the hypothesis that differences in SCFA concentrations are
associated with colorectal cancer.

\newpage

\subsection{Discussion}\label{discussion}

The observations from this study do not support the hypothesis that SCFA
concentrations are lower in individuals with tumors. Whether we directly
measured the SCFA concentration or investigated genes associated with
their production, no difference could be identified {[}Figure 1 \& 3{]}.
Further, no benefit could be found in using the concentrations to help
classify individuals with and without tumors {[}Figure 2{]}. These
observations are in stark contrast to some of the previous literature.

Much of the previous research on SCFA benefit to human CRC has been
illustrated in model systems (14). Many SCFAs are produced through the
breakdown of fiber (12) and a recent study in mice found that fiber
supplementation increased SCFA concentrations and decreased tumor burden
(16). Additionally, SCFAs such as butyrate can inhibit tumor growth in
\emph{in vitro} experiments (15). Yet, observations in humans has been
mixed. Previous case/control studies found associations with lower SCFA
concentrations in individuals with carcinoma tumors (17). However,
individual randomized controlled trials and a recent meta-analysis on
fiber supplementation to prevent tumor recurrence has found no benefit
(18, 21). Our results align with what has been reported in
randomized-controlled trials, that SCFAs do not provide general
protection against colorectal tumors. It is possible though that there
are specific instances where SCFAs may be beneficial.

One limitation of ours and others research into the effect of SCFAs in
CRC has been that all tumors are treated as the same type. However,
there are known differences in the types of mutations that occur (22)
and treating all tumors as equal may actually hide any benefit that
could be found in specific subsets of individuals. Similar to the idea
of specific immunotherapy for specific tumors (23), SCFAs may have
beneficial effects for specific types of colorectal tumors. Future
research will need to test if this is a valid hypothesis. Although there
are limitations with the current research on SCFAs and colorectal
tumors, evidence is building that these specific metabolites may not be
protective. Yet, taxa that are associated with SCFA production are
consistently higher in normal colons than those with carcinomas (7, 8,
24).

The potential protection against colorectal cancers may not be from
SCFAs even though taxa associated with their production are higher in
individuals with normal colons versus patients with carcinomas (7, 8,
24). Protection could be via a different pathway and by extension other
metabolites that have not been extensively studied. Alternatively,
protection may not occur via a metabolite but instead through niche
exclusion of mouth-associated microbes (e.g. \emph{Fusobacterium},
\emph{Porphyromonas}, \emph{Parvimonas}, \emph{Peptostreptococcus} (9,
10)). The idea of niche exclusion is similar to how the community
protects against \emph{Clostridium difficile} infection (25) with
chronic inflammation replacing the role of antibiotics. Although we did
not find lower concentrations of SCFAs associated with colorectal
tumors, we think that there are many exciting new avenues to explore
because of these results

\newpage

\subsection{Conclusions}\label{conclusions}

Our observations found no difference in SCFA concentration, their
utility as a classification tool, and for genes for enzymes involved in
SCFA synthesis between individuals with normal colons and patients with
either adenoma or carcinoma tumors. Although these results are different
than other reports in the literature, they do align with the randomized
controlled trials that have tested fiber use in preventing colorectal
tumor recurrence. Overall, these results suggest that the SCFAs produced
by resident microbes do not protect against tumor. By focusing on other
types of metabolites, the identification of more promising therapeutic
options for use in treating colorectal cancer may be found.

\newpage

\subsection{Materials and Methods}\label{materials-and-methods}

\textbf{Study design and sampling.} The overall protocol has been
described in detail previously (26, 27). In brief, this study used fecal
samples obtained at either a single cross-sectional time point (n=490)
or from before (pre-) and after (post-) treatment of a patient's tumor
(n=67). For patients undergoing treatment for their tumor the length of
time between their initial and follow up sample ranged from 188 - 546
days. Our use of treatment has been previously defined as encompassing
removal of a tumor with or without chemotherapy and radiation (26).
Diagnosis of tumor was made by colonoscopic examination and
histopathological review of biopsies obtained (26, 27). The University
of Michigan Institutional Review Board approved the study and informed
consent was obtained from all participants in accordance to the
guidelines set out by the Helsinki Declaration.

\textbf{Measuring specific SCFAs.} Our protocol for the measurement of
acetate, butyrate, and propionate followed a previously published
protocol that used a High-Performance Liquid Chromatography (HPLC)
machine (28). The following changes to this protocol included the use of
frozen fecal samples suspended in 1ml of PBS instead of fecal
suspensions in DNA Genotek OmniGut tubes, and the use of the actual
weight of fecal samples instead of the average weight for SCFA
concentration normalizations. These methodological changes did not
affect the overall median concentrations of these SCFAs between the two
studies (see Table 1 (28) and Figure 1 here).

\textbf{16s rRNA gene sequencing.} The workflow and processing have been
previously described (26, 29, 30). The major differences from these
previous reports include: the use of version 1.39.5 of the mothur
software package and clustering Operational Taxonomic Units (OTUs) at
97\% similarity using the OptClust algorithm (31).

\textbf{Generating imputed metagenomes.} The use of PICRUSt version
1.1.2 with the recommended standard operating protocol (20) was used.
Briefly, the mothur shared file and metadata was converted into a biom
formatted table using the biom convert function, the subsequent biom
file was processed with the `normalize\_by\_copy\_number.py' function,
and subsequent imputed metagenomes created using the
`predict\_metagenomes.py' function.

\textbf{Obtaining Operational Protein Families from metagenomes.} A
subset of the cross-sectional group (n=490) containing a total of 85
individuals (normal n=29, adenoma n=28, and carcinoma n=28) was shotgun
sequenced on an Illumina HiSeq using 125 bp paired end reads and a
previously described method (32). Briefly, the sequences were quality
filtered and sequences aligning to the human genome were removed prior
to contig assembly with MEGAHIT (33). Open Reading Frames (ORFs) were
identified using Prodigal (34), counts generated using Diamond (35),
subsequent clustering into Operational Protein Families (OPFs) used
mmseq2 (36), and OPF alignment used the KEGG database (37).

\textbf{Pulling genes involved with SCFA synthesis.} Specific genes
located near the end of the pathways involved in the synthesis of
acetate, butyrate, and propionate were analyzed for any differences
between individuals with normal colons and those with tumors. These
genes were based on pathways from KEGG as well as previous research (37,
38) and a list can be found in the supplemental material {[}Table S1{]}.

\textbf{Random Forest models.} The model was first trained on 80\% of
the data and then tested on the held out 20\% (80/20 split) using the
Random Forest algorithm for classification models (19). This was
repeated on 100 different 80/20 splits of the data to generate a
reasonable range for the AUC of the model. The reported AUCs, unless
otherwise specified, are for the test sets. The classification models
were built to group normal versus adenoma, normal versus carcinoma, and
high versus low SCFA concentrations.

\textbf{Statistical analysis workflow.} All analysis was performed using
the statistical language R (39). Generally, a Kruskal-Walis rank sum
test with a Dunn's post-hoc test was used to assess differences between
the groups used. Where appropriate Benjamini-Hochberg was used to
correct for multiple comparisons (40). First, we assessed differences in
SCFA concentrations measured by HPLC between individuals with normal
colons and patients with tumors (adenoma or carcinoma). We then analyzed
whether SCFA concentrations changed in patients with an adenoma or
carcinoma pre- versus post-treatment. Next, we assessed whether OTUs
alone or OTUs and SCFAs were better able to classify individuals with
and without tumor using Random Forest models. Next, the imputed gene
counts of important mediators of SCFA synthesis was tested.
Additionally, the counts generated for OPFs that matched important genes
involved with SCFA creation were analyzed. Finally, models to classify
high or low SCFA concentration based on the median of each SCFA and 16S
rRNA gene sequencing data was created using the Random Forest algorithm.

\newpage

\subsection{Acknowledgements}\label{acknowledgements}

The authors thank the Great Lakes-New England Early Detection Research
Network for providing the fecal samples that were used in this study. We
would also like to thank Kwi Kim and Thomas M Schmidt for their help in
running the short-chain fatty acid analysis on the High-Performance
Liquid Chromatography machine at the University of Michigan. Salary
support for Marc A. Sze came from the Canadian Institute of Health
Research and NIH grant UL1TR002240. Salary support for Patrick D.
Schloss came from NIH grants P30DK034933 and 1R01CA215574.

\newpage

\subsection{References}\label{references}

\hypertarget{refs}{}
\hypertarget{ref-Haggar2009}{}
1. \textbf{Haggar F}, \textbf{Boushey R}. 2009. Colorectal cancer
epidemiology: Incidence, mortality, survival, and risk factors. Clinics
in Colon and Rectal Surgery \textbf{22}:191--197.
doi:\href{https://doi.org/10.1055/s-0029-1242458}{10.1055/s-0029-1242458}.

\hypertarget{ref-Siegel2016}{}
2. \textbf{Siegel RL}, \textbf{Miller KD}, \textbf{Jemal A}. 2016.
Cancer statistics, 2016. CA: A Cancer Journal for Clinicians
\textbf{66}:7--30.
doi:\href{https://doi.org/10.3322/caac.21332}{10.3322/caac.21332}.

\hypertarget{ref-Lichtenstein2000}{}
3. \textbf{Lichtenstein P}, \textbf{Holm NV}, \textbf{Verkasalo PK},
\textbf{Iliadou A}, \textbf{Kaprio J}, \textbf{Koskenvuo M},
\textbf{Pukkala E}, \textbf{Skytthe A}, \textbf{Hemminki K}. 2000.
Environmental and heritable factors in the causation of cancer analyses
of cohorts of twins from sweden, denmark, and finland. New England
Journal of Medicine \textbf{343}:78--85.
doi:\href{https://doi.org/10.1056/nejm200007133430201}{10.1056/nejm200007133430201}.

\hypertarget{ref-Zackular2013}{}
4. \textbf{Zackular JP}, \textbf{Baxter NT}, \textbf{Iverson KD},
\textbf{Sadler WD}, \textbf{Petrosino JF}, \textbf{Chen GY},
\textbf{Schloss PD}. 2013. The gut microbiome modulates colon
tumorigenesis. mBio \textbf{4}:e00692--13--e00692--13.
doi:\href{https://doi.org/10.1128/mbio.00692-13}{10.1128/mbio.00692-13}.

\hypertarget{ref-Baxter2014}{}
5. \textbf{Baxter NT}, \textbf{Zackular JP}, \textbf{Chen GY},
\textbf{Schloss PD}. 2014. Structure of the gut microbiome following
colonization with human feces determines colonic tumor burden.
Microbiome \textbf{2}:20.
doi:\href{https://doi.org/10.1186/2049-2618-2-20}{10.1186/2049-2618-2-20}.

\hypertarget{ref-Zackular2015}{}
6. \textbf{Zackular JP}, \textbf{Baxter NT}, \textbf{Chen GY},
\textbf{Schloss PD}. 2015. Manipulation of the gut microbiota reveals
role in colon tumorigenesis. mSphere \textbf{1}:e00001--15.
doi:\href{https://doi.org/10.1128/msphere.00001-15}{10.1128/msphere.00001-15}.

\hypertarget{ref-Shah2017}{}
7. \textbf{Shah MS}, \textbf{DeSantis TZ}, \textbf{Weinmaier T},
\textbf{McMurdie PJ}, \textbf{Cope JL}, \textbf{Altrichter A},
\textbf{Yamal J-M}, \textbf{Hollister EB}. 2017. Leveraging
sequence-based faecal microbial community survey data to identify a
composite biomarker for colorectal cancer. Gut \textbf{67}:882--891.
doi:\href{https://doi.org/10.1136/gutjnl-2016-313189}{10.1136/gutjnl-2016-313189}.

\hypertarget{ref-meta_analysis_crc_Sze2018}{}
8. \textbf{Sze MA}, \textbf{Schloss PD}. 2018. Leveraging existing 16S
rRNA gene surveys to identify reproducible biomarkers in individuals
with colorectal tumors.
doi:\href{https://doi.org/10.1101/285486}{10.1101/285486}.

\hypertarget{ref-Zeller2014}{}
9. \textbf{Zeller G}, \textbf{Tap J}, \textbf{Voigt AY},
\textbf{Sunagawa S}, \textbf{Kultima JR}, \textbf{Costea PI},
\textbf{Amiot A}, \textbf{Bohm J}, \textbf{Brunetti F},
\textbf{Habermann N}, \textbf{Hercog R}, \textbf{Koch M},
\textbf{Luciani A}, \textbf{Mende DR}, \textbf{Schneider MA},
\textbf{Schrotz-King P}, \textbf{Tournigand C}, \textbf{Nhieu JTV},
\textbf{Yamada T}, \textbf{Zimmermann J}, \textbf{Benes V},
\textbf{Kloor M}, \textbf{Ulrich CM}, \textbf{Knebel Doeberitz M von},
\textbf{Sobhani I}, \textbf{Bork P}. 2014. Potential of fecal microbiota
for early-stage detection of colorectal cancer. Molecular Systems
Biology \textbf{10}:766--766.
doi:\href{https://doi.org/10.15252/msb.20145645}{10.15252/msb.20145645}.

\hypertarget{ref-Baxter2016}{}
10. \textbf{Baxter NT}, \textbf{Ruffin MT}, \textbf{Rogers MAM},
\textbf{Schloss PD}. 2016. Microbiota-based model improves the
sensitivity of fecal immunochemical test for detecting colonic lesions.
Genome Medicine \textbf{8}.
doi:\href{https://doi.org/10.1186/s13073-016-0290-3}{10.1186/s13073-016-0290-3}.

\hypertarget{ref-Kostic2011}{}
11. \textbf{Kostic AD}, \textbf{Gevers D}, \textbf{Pedamallu CS},
\textbf{Michaud M}, \textbf{Duke F}, \textbf{Earl AM}, \textbf{Ojesina
AI}, \textbf{Jung J}, \textbf{Bass AJ}, \textbf{Tabernero J},
\textbf{Baselga J}, \textbf{Liu C}, \textbf{Shivdasani RA},
\textbf{Ogino S}, \textbf{Birren BW}, \textbf{Huttenhower C},
\textbf{Garrett WS}, \textbf{Meyerson M}. 2011. Genomic analysis
identifies association of fusobacterium with colorectal carcinoma.
Genome Research \textbf{22}:292--298.
doi:\href{https://doi.org/10.1101/gr.126573.111}{10.1101/gr.126573.111}.

\hypertarget{ref-Holscher2017}{}
12. \textbf{Holscher HD}. 2017. Dietary fiber and prebiotics and the
gastrointestinal microbiota. Gut Microbes \textbf{8}:172--184.
doi:\href{https://doi.org/10.1080/19490976.2017.1290756}{10.1080/19490976.2017.1290756}.

\hypertarget{ref-Louis2016}{}
13. \textbf{Louis P}, \textbf{Flint HJ}. 2016. Formation of propionate
and butyrate by the human colonic microbiota. Environmental Microbiology
\textbf{19}:29--41.
doi:\href{https://doi.org/10.1111/1462-2920.13589}{10.1111/1462-2920.13589}.

\hypertarget{ref-test_OKeefe2016}{}
14. \textbf{O'Keefe SJD}. 2016. Diet, microorganisms and their
metabolites and colon cancer. Nature Reviews Gastroenterology \&
Hepatology \textbf{13}:691--706.
doi:\href{https://doi.org/10.1038/nrgastro.2016.165}{10.1038/nrgastro.2016.165}.

\hypertarget{ref-Encarnao2018}{}
15. \textbf{Encarnação JC}, \textbf{Pires AS}, \textbf{Amaral RA},
\textbf{Gonçalves TJ}, \textbf{Laranjo M}, \textbf{Casalta-Lopes JE},
\textbf{Gonçalves AC}, \textbf{Sarmento-Ribeiro AB}, \textbf{Abrantes
AM}, \textbf{Botelho MF}. 2018. Butyrate, a dietary fiber derivative
that improves irinotecan effect in colon cancer cells. The Journal of
Nutritional Biochemistry \textbf{56}:183--192.
doi:\href{https://doi.org/10.1016/j.jnutbio.2018.02.018}{10.1016/j.jnutbio.2018.02.018}.

\hypertarget{ref-Bishehsari2018}{}
16. \textbf{Bishehsari F}, \textbf{Engen P}, \textbf{Preite N},
\textbf{Tuncil Y}, \textbf{Naqib A}, \textbf{Shaikh M}, \textbf{Rossi
M}, \textbf{Wilber S}, \textbf{Green S}, \textbf{Hamaker B},
\textbf{Khazaie K}, \textbf{Voigt R}, \textbf{Forsyth C},
\textbf{Keshavarzian A}. 2018. Dietary fiber treatment corrects the
composition of gut microbiota, promotes SCFA production, and suppresses
colon carcinogenesis. Genes \textbf{9}:102.
doi:\href{https://doi.org/10.3390/genes9020102}{10.3390/genes9020102}.

\hypertarget{ref-Ohigashi2013}{}
17. \textbf{Ohigashi S}, \textbf{Sudo K}, \textbf{Kobayashi D},
\textbf{Takahashi O}, \textbf{Takahashi T}, \textbf{Asahara T},
\textbf{Nomoto K}, \textbf{Onodera H}. 2013. Changes of the intestinal
microbiota, short chain fatty acids, and fecal pH in patients with
colorectal cancer. Digestive Diseases and Sciences
\textbf{58}:1717--1726.
doi:\href{https://doi.org/10.1007/s10620-012-2526-4}{10.1007/s10620-012-2526-4}.

\hypertarget{ref-Yao2017}{}
18. \textbf{Yao Y}, \textbf{Suo T}, \textbf{Andersson R}, \textbf{Cao
Y}, \textbf{Wang C}, \textbf{Lu J}, \textbf{Chui E}. 2017. Dietary fibre
for the prevention of recurrent colorectal adenomas and carcinomas.
Cochrane Database of Systematic Reviews.
doi:\href{https://doi.org/10.1002/14651858.cd003430.pub2}{10.1002/14651858.cd003430.pub2}.

\hypertarget{ref-randomforest_citation_2002}{}
19. \textbf{Liaw A}, \textbf{Wiener M}. 2002. Classification and
regression by randomForest. R News \textbf{2}:18--22.

\hypertarget{ref-Langille2013}{}
20. \textbf{Langille MGI}, \textbf{Zaneveld J}, \textbf{Caporaso JG},
\textbf{McDonald D}, \textbf{Knights D}, \textbf{Reyes JA},
\textbf{Clemente JC}, \textbf{Burkepile DE}, \textbf{Thurber RLV},
\textbf{Knight R}, \textbf{Beiko RG}, \textbf{Huttenhower C}. 2013.
Predictive functional profiling of microbial communities using 16S rRNA
marker gene sequences. Nature Biotechnology \textbf{31}:814--821.
doi:\href{https://doi.org/10.1038/nbt.2676}{10.1038/nbt.2676}.

\hypertarget{ref-Schatzkin2000}{}
21. \textbf{Schatzkin A}, \textbf{Lanza E}, \textbf{Corle D},
\textbf{Lance P}, \textbf{Iber F}, \textbf{Caan B}, \textbf{Shike M},
\textbf{Weissfeld J}, \textbf{Burt R}, \textbf{Cooper MR},
\textbf{Kikendall JW}, \textbf{Cahill J}, \textbf{Freedman L},
\textbf{Marshall J}, \textbf{Schoen RE}, \textbf{Slattery M}. 2000. Lack
of effect of a low-fat, high-fiber diet on the recurrence of colorectal
adenomas. New England Journal of Medicine \textbf{342}:1149--1155.
doi:\href{https://doi.org/10.1056/nejm200004203421601}{10.1056/nejm200004203421601}.

\hypertarget{ref-Fearon1990}{}
22. \textbf{Fearon ER}, \textbf{Vogelstein B}. 1990. A genetic model for
colorectal tumorigenesis. Cell \textbf{61}:759--767.
doi:\href{https://doi.org/10.1016/0092-8674(90)90186-i}{10.1016/0092-8674(90)90186-i}.

\hypertarget{ref-Thomas2016}{}
23. \textbf{Thomas X}, \textbf{Heiblig M}. 2016. The development of
agents targeting the BCR-ABL tyrosine kinase as philadelphia
chromosome-positive acute lymphoblastic leukemia treatment. Expert
Opinion on Drug Discovery \textbf{11}:1061--1070.
doi:\href{https://doi.org/10.1080/17460441.2016.1227318}{10.1080/17460441.2016.1227318}.

\hypertarget{ref-normalization_Sze2017}{}
24. \textbf{Sze MA}, \textbf{Baxter NT}, \textbf{Ruffin MT},
\textbf{Rogers MAM}, \textbf{Schloss PD}. 2017. Normalization of the
microbiota in patients after treatment for colonic lesions. Microbiome
\textbf{5}.
doi:\href{https://doi.org/10.1186/s40168-017-0366-3}{10.1186/s40168-017-0366-3}.

\hypertarget{ref-Theriot2015}{}
25. \textbf{Theriot CM}, \textbf{Young VB}. 2015. Interactions between
the gastrointestinal microbiome and clostridium difficile. Annual Review
of Microbiology \textbf{69}:445--461.
doi:\href{https://doi.org/10.1146/annurev-micro-091014-104115}{10.1146/annurev-micro-091014-104115}.

\hypertarget{ref-normalization_sze2017}{}
26. \textbf{Sze MA}, \textbf{Baxter NT}, \textbf{Ruffin MT},
\textbf{Rogers MAM}, \textbf{Schloss PD}. 2017. Normalization of the
microbiota in patients after treatment for colonic lesions. Microbiome
\textbf{5}.
doi:\href{https://doi.org/10.1186/s40168-017-0366-3}{10.1186/s40168-017-0366-3}.

\hypertarget{ref-crc_model_baxter2016}{}
27. \textbf{Baxter NT}, \textbf{Ruffin MT}, \textbf{Rogers MAM},
\textbf{Schloss PD}. 2016. Microbiota-based model improves the
sensitivity of fecal immunochemical test for detecting colonic lesions.
Genome Medicine \textbf{8}.
doi:\href{https://doi.org/10.1186/s13073-016-0290-3}{10.1186/s13073-016-0290-3}.

\hypertarget{ref-scfa_measures_venkataraman2016}{}
28. \textbf{Venkataraman A}, \textbf{Sieber JR}, \textbf{Schmidt AW},
\textbf{Waldron C}, \textbf{Theis KR}, \textbf{Schmidt TM}. 2016.
Variable responses of human microbiomes to dietary supplementation with
resistant starch. Microbiome \textbf{4}.
doi:\href{https://doi.org/10.1186/s40168-016-0178-x}{10.1186/s40168-016-0178-x}.

\hypertarget{ref-Schloss2009}{}
29. \textbf{Schloss PD}, \textbf{Westcott SL}, \textbf{Ryabin T},
\textbf{Hall JR}, \textbf{Hartmann M}, \textbf{Hollister EB},
\textbf{Lesniewski RA}, \textbf{Oakley BB}, \textbf{Parks DH},
\textbf{Robinson CJ}, \textbf{Sahl JW}, \textbf{Stres B},
\textbf{Thallinger GG}, \textbf{Horn DJV}, \textbf{Weber CF}. 2009.
Introducing mothur: Open-source, platform-independent,
community-supported software for describing and comparing microbial
communities. Applied and Environmental Microbiology
\textbf{75}:7537--7541.
doi:\href{https://doi.org/10.1128/aem.01541-09}{10.1128/aem.01541-09}.

\hypertarget{ref-Kozich2013}{}
30. \textbf{Kozich JJ}, \textbf{Westcott SL}, \textbf{Baxter NT},
\textbf{Highlander SK}, \textbf{Schloss PD}. 2013. Development of a
dual-index sequencing strategy and curation pipeline for analyzing
amplicon sequence data on the MiSeq illumina sequencing platform.
Applied and Environmental Microbiology \textbf{79}:5112--5120.
doi:\href{https://doi.org/10.1128/aem.01043-13}{10.1128/aem.01043-13}.

\hypertarget{ref-opticlust_westcott2017}{}
31. \textbf{Westcott SL}, \textbf{Schloss PD}. 2017. OptiClust, an
improved method for assigning amplicon-based sequence data to
operational taxonomic units. mSphere \textbf{2}:e00073--17.
doi:\href{https://doi.org/10.1128/mspheredirect.00073-17}{10.1128/mspheredirect.00073-17}.

\hypertarget{ref-Hannigan2017}{}
32. \textbf{Hannigan GD}, \textbf{Duhaime MB}, \textbf{Ruffin MT},
\textbf{Koumpouras CC}, \textbf{Schloss PD}. 2017. Diagnostic potential
\& the interactive dynamics of the colorectal cancer virome.
doi:\href{https://doi.org/10.1101/152868}{10.1101/152868}.

\hypertarget{ref-Li2015}{}
33. \textbf{Li D}, \textbf{Liu C-M}, \textbf{Luo R}, \textbf{Sadakane
K}, \textbf{Lam T-W}. 2015. MEGAHIT: An ultra-fast single-node solution
for large and complex metagenomics assembly via succinct de bruijn
graph. Bioinformatics \textbf{31}:1674--1676.
doi:\href{https://doi.org/10.1093/bioinformatics/btv033}{10.1093/bioinformatics/btv033}.

\hypertarget{ref-Hyatt2010}{}
34. \textbf{Hyatt D}, \textbf{Chen G-L}, \textbf{LoCascio PF},
\textbf{Land ML}, \textbf{Larimer FW}, \textbf{Hauser LJ}. 2010.
Prodigal: Prokaryotic gene recognition and translation initiation site
identification. BMC Bioinformatics \textbf{11}:119.
doi:\href{https://doi.org/10.1186/1471-2105-11-119}{10.1186/1471-2105-11-119}.

\hypertarget{ref-Buchfink2014}{}
35. \textbf{Buchfink B}, \textbf{Xie C}, \textbf{Huson DH}. 2014. Fast
and sensitive protein alignment using DIAMOND. Nature Methods
\textbf{12}:59--60.
doi:\href{https://doi.org/10.1038/nmeth.3176}{10.1038/nmeth.3176}.

\hypertarget{ref-Steinegger2017}{}
36. \textbf{Steinegger M}, \textbf{Söding J}. 2017. MMseqs2 enables
sensitive protein sequence searching for the analysis of massive data
sets. Nature Biotechnology.
doi:\href{https://doi.org/10.1038/nbt.3988}{10.1038/nbt.3988}.

\hypertarget{ref-Kanehisa2015}{}
37. \textbf{Kanehisa M}, \textbf{Sato Y}, \textbf{Kawashima M},
\textbf{Furumichi M}, \textbf{Tanabe M}. 2015. KEGG as a reference
resource for gene and protein annotation. Nucleic Acids Research
\textbf{44}:D457--D462.
doi:\href{https://doi.org/10.1093/nar/gkv1070}{10.1093/nar/gkv1070}.

\hypertarget{ref-scfa_baxter2014}{}
38. \textbf{Baxter NT}, \textbf{Zackular JP}, \textbf{Chen GY},
\textbf{Schloss PD}. 2014. Structure of the gut microbiome following
colonization with human feces determines colonic tumor burden.
Microbiome \textbf{2}:20.
doi:\href{https://doi.org/10.1186/2049-2618-2-20}{10.1186/2049-2618-2-20}.

\hypertarget{ref-r_citation_2017}{}
39. \textbf{R Core Team}. 2017. R: A language and environment for
statistical computing. R Foundation for Statistical Computing, Vienna,
Austria.

\hypertarget{ref-benjamini_controlling_1995}{}
40. \textbf{Benjamini Y}, \textbf{Hochberg Y}. 1995. Controlling the
false discovery rate: A practical and powerful approach to multiple
testing. Journal of the Royal Statistical Society Series B
(Methodological) \textbf{57}:289--300.

\newpage

\textbf{Figure 1. No change in SCFA measurements was observed between
normal, adenoma, and carcinoma individuals using HPLC.} Acetate
concentrations in fecal samples of individuals with normal colons,
adenomas, and carcinomas (A). Butyrate concentrations in fecal samples
of individuals with normal colons, adenomas, and carcinomas (B).
Propionate concentrations in fecal samples of individuals with normal
colons, adenomas, and carcinomas (C). The black lines indicate the
median SCFA concentration. Acetate concentrations in fecal samples
before and after treatment for adenoma (yellow) and carcinoma (red) (D).
Butyrate concentrations in fecal samples before and after treatment for
adenoma (yellow) and carcinoma (red) (E). Propionate concentrations in
fecal samples before and after treatment for adenoma (yellow) and
carcinoma (red) (F). The black dots and lines represent the median
change in SCFA concentration.

\textbf{Figure 2. SCFAs do not improve OTU-based Random Forest models.}
The area under the curve of 100 different 80/20 split OTU-based normal
versus adenoma 10-fold CV models with and without SCFAs (A). The area
under the curve of 100 different 80/20 OTU-based normal versus carcinoma
10-fold CV models with and without SCFAs (B). The black line represents
the median AUC. The dotted line highlights an AUC of 0.5.

\textbf{Figure 3. No change in butyrate producing genes identified
between normal, adenoma, and carcinoma individuals.} Imputed gene
relative abundance of important butyrate pathway genes using PICRUSt
(A). Counts per million (corrected for size and number of contigs in an
OPF) for the Butyrate Kinase gene (B). The other butyrate pathway genes
from the PICRUSt analysis did not align to any of the OPFs in the
metagenome analysis.

\newpage

\textbf{Figure S1. OTU-based Random Forest models of SCFA
concentrations.} The train and test results of 100 different 80/20
OTU-based models with 10-fold CV based on higher or lower than the
median SCFA concentration using classification Random Forest (A). The
top 10 OTUs based on mean decrease in accuracy (MDA) for each model,
colored by their lowest taxonomic identification (B).


\end{document}
