---
title: "Revisiting the Relationship between Short-Chain Fatty Acids, the Microbiota, and Colorectal Tumors"
bibliography: references.bib
output:
  pdf_document:
    keep_tex: true
    includes:
      in_header: header.tex
csl: mbio.csl 
fontsize: 11pt
geometry: margin=1.0in
---


```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE, message=FALSE, warning=FALSE}
source('../code/functions.R')

loadLibs(c("tidyverse", "knitr", "rmarkdown"))
```


```{r rdata_tables, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

# Read in meta data
metaf <- read_csv("../data/raw/metadata/good_metaf_final.csv")
metai <- read_csv("../data/raw/metadata/metaI_final.csv")
geof_samples <- read_tsv("../data/raw/metadata/303030_subset/NexteraXT003Map.tsv", 
                         col_names = F) %>% filter(X22 != "Negative")

# Read in analysis of SCFA by HPLC
hplc_groups_scfa_kruskal <- read_csv("../data/process/tables/acetate_kruskal_crc_groups.csv") %>% 
  mutate(scfa = "acetate") %>% slice(1) %>% 
  bind_rows(
    read_csv("../data/process/tables/butyrate_kruskal_crc_groups.csv") %>% 
      mutate(scfa = "butyrate") %>% slice(1), 
    read_csv("../data/process/tables/propionate_kruskal_crc_groups.csv") %>% 
      mutate(scfa = "propionate") %>% slice(1)) %>% 
  mutate(bh = p.adjust(pvalue, method = "BH"))

hplc_treat_scfa_kruskal <- read_csv("../data/process/tables/scfa_treatment_comparison_pvalues.csv") %>% 
  filter(scfa != "isobutyrate")

# Load in Random Forest with and without SCFA summary data
disease_rf <- read_csv("../data/process/tables/rf_model_ttest_comparisons.csv")

# Load in data for select genes in SCFA pathways
picrust_select <- read_csv("../data/process/tables/specific_scfa_kruskal_picrust_summary.csv")
picrust_select_treat <- read_csv("../data/process/tables/specific_scfa_wilcox_picrust_treatment_summary.csv")
opf_select <- read_csv("../data/process/tables/select_scfa_opf_kruskal_summary.csv")

# Load in the group comparison from SCFA models
class_group_rf_data <- read_csv("../data/process/tables/full_model_class_group_results.csv")
reg_group_rf_data <- read_csv("../data/process/tables/full_model_reg_group_results.csv")

```


\vspace{35mm}

Running title: SCFAs and colorectal tumors

\vspace{35mm}


Marc A Sze${^1}$, Nicholas A Lesniak${^1}$, Mack T Ruffin IV${^2}$, Patrick D. Schloss${^1}$${^\dagger}$

\vspace{40mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI 48109

$2$ Department of Family Medicine and Community Medicine, Penn State Hershey Medical Center, Hershey, PA


\newpage
\linenumbers


## Abstract

**Background.** Colorectal cancer (CRC) is a growing health concern with the majority of the risk for developing disease being due to environmental factors. The microbiota is one of these environmental factors with certain bacterial community members being associated with CRC, while other taxa are associated to colons without tumors. Some of the taxa associated to colons without tumors can use fiber to produce short-chain fatty acids (SCFAs) that can inhibit tumor growth in model systems. However, the data supporting the importance of SCFAs in human CRC is less certain. Here, we test the hypothesis that SCFA concentrations are different in individuals with colorectal tumors. 

**Methods.** We analyzed a cross-sectional (n=`r length(metai$sample)`) and longitudinal pre- and post-treatment (n=`r length(metaf$initial)`) group for their concentrations of acetate, butyrate, and propionate. Analysis also included tumor classification models using Random Forest, imputed gene relative abundance with PICRUSt, and metagenomic sequencing on a subset (n=`r length(geof_samples$X30)`) of the total cross-sectional group.  

**Results.** No difference in SCFA concentrations were found between individuals without tumors and patients with adenomas or carcinomas (P-value `r ifelse(min(pull(hplc_groups_scfa_kruskal, bh)) > 0.15, invisible(">"), invisible("<"))` 0.15). There was no difference in classification models with or without SCFAs in their ability to predict patients with adenomas or carcinomas versus individuals without tumors (P-value `r ifelse(min(pull(disease_rf, bh)) > 0.05, invisible(">"), invisible("<"))` 0.05). Using metagenomic sequencing, there was also no difference in genes involved with SCFA synthesis between individuals without tumors and patients with adenomas or carcinomas (P-value `r ifelse(min(pull(opf_select, bh)) > 0.70, invisible(">"), invisible("<"))` 0.70). 

**Conclusions.** Although our data does not support the hypothesis that SCFAs are different in individuals that have colorectal tumors, there may be context specific scenarios where SCFAs may still be beneficial for treatment of CRC. Alternatively, there may be other mechanisms that have not been thoroughly investigated that are more important to the development of human CRC. 


\newpage

## Introduction

Colorectal cancer (CRC) is currently the third highest cancer-related cause of death within the US [@Haggar2009; @Siegel2016]. Although there is a genetic component to the disease, the environment is attributed to being a larger risk factor for CRC [@Lichtenstein2000]. These environmental risk factors include but are not limited to smoking cigarettes, diet, and the microbiota [@FlissIsakov2017; @Lee2015; @Kostic2011]. Many of these environmental risk factors are capable of being modified, and this has lead to the investigation of how the microbiota may exacerbate or cause tumorigensis [@Zackular2013; @Baxter2014; @Zackular2015] and whether the bacterial community is altered [@Shah2017; @meta_analysis_crc_Sze2018]. Multiple reports in case/control studies have identified bacterial taxa commonly associated with individuals without tumors to be decreased in patients with carcinoma tumors [@Zeller2014; @Baxter2016; @meta_analysis_crc_Sze2018]. Many of these taxa within individuals wihtout tumors actively produce short-chain fatty acids (SCFAs) from fiber that are a part of our general diet [@Holscher2017]. The most extensively studied of these SCFAs are acetate, butyrate, and propionate [@Louis2016]. Overall, the specific bacterial taxa of the microbiota that create SCFAs are an attractive target to modulate the risk of CRC.

Specific SCFAs, like butyrate, have shown positive results for CRC treatment within model systems [@test_OKeefe2016]. Butyrate has been shown to inhibit cancer cell growth in *in vitro* systems [@Encarnao2018]. Additionally, supplementation with food sources that bacteria use to create these SCFAs may also be able to confer beneficial effects. For example, fiber supplementation in mouse models of CRC caused an overall reduction in tumor burden while also increasing SCFA concentrations [@Bishehsari2018]. Although these model systems provide important preliminary evidence towards the ability of SCFAs to reduce and treat tumors, the studies reporting benefit in humans has been less convincing.  

There is a lack of evidence on the benefit of increasing SCFA concentrations to protect against CRC in human populations. The initial case/control studies that investigated SCFA concentrations in CRC found that patients with carcinomas had lower concentrations of acetate, butyrate, and propionate versus either patients with adenomas or individuals without colon tumors [@Ohigashi2013]. Although this would argue that increasing SCFA concentrations could be protective against tumorigenesis, fiber supplementation in randomized controlled trials have consistently failed to protect against tumor recurrence [@Yao2017]. These findings would argue against the utility of treatments that aim to use SCFAs to reduce or protect against tumorigenesis. Given the lack of clear evidence in human studies of the benefit of SCFAs in CRC, there is a need for more investigation into this area.
  
Our study fills some of the current gaps in the literature that relate to the study of SCFAs and CRC in human populations. Specifically, it tests previous case/control findings on SCFA concentrations in individuals with and without tumors. We also test previous suggestions that there is a continuous reduction in SCFA concentrations as tumor severity increases by increasing the number of patients with adenomas in our study. Additionally, we build upon these observations and assesses the utility of using SCFAs and Operational Taxonomic Units (OTUs) as a risk stratification tool of colorectal tumors (adenoma or carcinoma). Collectively, this study provides important information on the replicability of previous findings in humans by extensively studying how SCFAs are associated with colorectal tumors.

To accomplish this task we directly measured the concentration of acetate, butyrate, and propionate within fecal samples for two different groups. The first group had a sample obtained at a single cross sectional point in time while the second group had samples obtained before (pre-) and after (post-)treatment for colorectal tumors. Additionally, we (i) assessed the affect adding SCFA concentrations to OTU data had on classification of patients with adenoma or carcinoma using the Random Forest algorithm [@randomforest_citation_2002], (ii) used PICRUSt [@Langille2013] and metagenomic sequencing to assess the presence of genes involved in SCFA synthesis, and (iii) analyzed how well 16S rRNA gene sequencing predicts SCFA concentrations. This investigation provides additional information as to whether SCFAs are decreased in patients with colorectal tumors and provides context as to whether targeting taxa to increase SCFA concentrations is a viable option to protect against colon tumorigenesis.


\newpage


## Results 

**Decreased SCFA concentrations are not associated with tumors.** We used high-performance liquid chromatography (HPLC) to measure acetate, butyrate, and propionate concentrations of frozen fecal samples from `r length(metai$sample)` individuals at a cross-sectional point in time. There was no difference between individuals without colon tumors (n=`r length(filter(metai, dx == "normal") %>% pull(sample))`) and patients with either an adenoma (n=`r length(filter(metai, dx == "adenoma") %>% pull(sample))`) or carcinoma (n=`r length(filter(metai, dx == "cancer") %>% pull(sample))`) for any of the SCFAs measured after multiple comparison correction (P-value `r ifelse(min(pull(hplc_groups_scfa_kruskal, bh)) > 0.15, invisible(">"), invisible("<"))` 0.15) [Figure 1A - 1C]. We next measured the concentration of SCFAs in `r length(metaf$initial)` patients with an adenoma (n=`r length(filter(metaf, dx == "adenoma") %>% pull(initial))`) or carcinoma (n=`r length(filter(metaf, dx == "cancer") %>% pull(initial))`) in which we had pre- and post-treatment fecal samples. Although there was a general trend for increasing acetate, butyrate, and propionate concentrations after treatment for tumors, there was no significant difference pre- and post-treatment for patients with adenomas (P-value `r ifelse(min(filter(hplc_treat_scfa_kruskal, disease == "adenoma") %>% pull(bh)) > 0.20, invisible(">"), invisible("<"))` 0.20) or carcinomas (P-value `r ifelse(min(filter(hplc_treat_scfa_kruskal, disease == "carcinoma") %>% pull(bh)) > 0.80, invisible(">"), invisible("<"))` 0.80) [Figure 1D - 1F]. 

**Changes in genes for enzymes involved in SCFA synthesis are not associated with tumors.** Using a list of specific genes that are important for the synthesis of SCFAs [Table S1], we looked for differences in gene abundance between individuals without colon tumors and patients with adenomas or carcinomas. First, using imputed gene relative abundance based on 16S rRNA gene sequencing we found no difference in any of the genes involved with acetate, butyrate, or propionate synthesis (P-value `r ifelse(min(pull(picrust_select, bh)) > 0.90, invisible(">"), invisible("<"))` 0.90) [Table S2]. This similarity between groups is highlighted by visualizing genes important in butyrate synthesis [Figure 2A]. Using a paired Wilcoxon rank-sum test, there also was no difference in imputed gene relative abundance between pre- and post-treatment samples for any genes involved with SCFA synthesis (P-value `r ifelse(min(pull(picrust_select_treat, bh)) > 0.70, invisible(">"), invisible("<"))` 0.70) [Table S3]. Next, we took a subset of these `r length(metai$sample)` fecal samples (n=`r length(geof_samples$X1)`) and used metagenomic sequencing to confirm these results. Like the imputed gene results, metagenomic analysis found that there was no difference in any of the genes involved in SCFA synthesis between individuals without colon tumors (n=`r length(filter(geof_samples, X30 == "Healthy") %>% pull(X2))`) and patients with adenoma (n=`r length(filter(geof_samples, X30 == "Adenoma") %>% pull(X2))`) or carcinoma (n=`r length(filter(geof_samples, X30 == "Cancer") %>% pull(X2))`) (P-value `r ifelse(min(pull(opf_select, bh)) > 0.70, invisible(">"), invisible("<"))` 0.70) [Table S4]. This lack of difference can be highlighted when we visualize the results for butyrate kinase [Figure 2B]. These observations provide evidence that gene prevalence for enzymes involved in SCFA synthesis does not change due to colorectal tumors. 


**Total significant positive correlations between OTU relative abundance and SCFA concentration were similar for individuals without tumors and patients with adenomas or carcinomas.** Having found no difference between individuals without tumors and patients with adenomas or carcinomas in SCFA concentrations or genes coding for enzymes involved with SCFA synthesis, we next investigated if specific OTUs correlated with SCFA concentrations. Using Spearmans rho we found that the majority of significant OTU correlations were to taxa from *Clostridiales*, *Lachnospiraceae*, and *Ruminococcaceae* [Figure 3 & Table S5]. A similar pattern was observed when using high/low SCFA groups based on the overall median concentration for that specific SCFA [Figure S1 & Table S6]. There was a noticeablely higher number of significant negative correlations associated with patients with adenomas for all SCFAs tested [Figure 3]. In particular, OTUs from the *Ruminococcaceae* family had the largest share of these negative correlations within patients with adenomas  [Figure 3]. Although patients with adenomas had more positive correlations between OTUs and SCFA concentrations, their total number was more similar to individuals without tumors or patients with carcinomas versus the analogous comparison for the number of negative correlations [Figure 3]. The number of positive correlations between OTUs and SCFA concentrations was similar between individuals without tumors and patients with a carcinoma [Figure 3]. Overall, these results suggest that the resident taxa that may change the most due to colon tumors may not be ones that are responsible for the production of acetate, butyrate, or propionate. 


**SCFA concentrations do not replace important *Clostridiales*, *Lachnospiraceae*, and *Ruminococcaceae* OTUs in Random Forest models built to classify tumors.** SCFA concentrations could improve prediction of tumors based on specific bacterial community structures. Our OTU data can be used in combination with SCFAs to assess whether there is a community dependent context to SCFA classification of tumors. Using the Random Forest algorithm we built models with OTU abundance data or OTU abundances and SCFA concentrations to classify normal versus adenoma and normal versus carcinoma fecal samples. For adenoma and carcinoma models, there was no difference between the median AUC of models with or without SCFA concentrations (P-value `r ifelse(min(pull(disease_rf, bh)) > 0.05, invisible(">"), invisible("<"))` 0.05) [Figure 4A & 4D]. There was little difference between the top 10 most important OTUs, as measured by mean decrease in accuracy, in models using SCFA and OTUs versus OTUs only [Figure 4B-C & 4E-F]. The main reason for this was because SCFA concentrations replaced the information gained by specific OTUs rather than add new information to the model [Figure 4B-C]. The SCFA concentrations also only replaced select OTUs with taxonomic classification to *Clostridiales*, *Lachnospiraceae*, and *Ruminococcaceae* with many OTUs within these taxa remaining in the top 10 [Figure 4B & 4C]. In combination with the previous results on taxa correlations, these observations provide additional evidence that the resident taxa within models that classify tumor are not ones associated with acetate, butyrate, or propionate production.     


**Random Forest models for SCFA concentrations have different top 10 most important OTUs than tumor models.** Using OTU data we built Random Forest models to classify either SCFA concentration or higher/lower than median SCFA concentrations. Overall, OTU data had a reasonable ability to classify both SCFA concentrations and high/low SCFA groups [Figure 5A & S2A]. However, these models tended to be over fit, suggesting that rarer taxa may be important for this classification [Figure 5A & S2A]. Additionally, there were differences in accuracy for both model types based on whether the individual had no tumors, an adenoma, or a carcinoma [Figure 5B & S2B]. There also was minimal overlap between these SCFA classification model's most important OTUs and those used to classify patients with adenoma or carcinoma tumors [Figure 4B-C, 4E-F, 5C-E, and S2C-E]. The only OTU that did overlap between the models was OTU00167 (*Clostridiales*) [Figure 4B-C, 4E-F, 5C-E, and S2C-E]. Additionally, OTU00167 was in the top 10 most important OTUs for the OTU adenoma model but not in the SCFA and OTU adenoma model while acetate and butyrate concentrations were [Figure 4B-C]. These observations provide further evidence that it is possible to identify specific OTUs associated with higher SCFA concentrations and that these OTUs belong to taxa known to produce acetate, butyrate, and propionate. Although it is possible to identify OTUs associated with SCFA production, our results do not support the hypothesis that SCFA concentration or OTUs associated with their production are different between individuals with no tumors and patients with adenomas or carcinomas.    


\newpage

## Discussion

The observations from this study do not support the hypothesis that SCFA concentrations are different in individuals with tumors. Whether we directly measured the SCFA concentration or investigated genes associated with their production, no difference could be identified between individuals without tumors and patients with adenomas or carcinomas [Figure 1 & 2]. Although there were differences in the number of significant correlations between SCFA concentration and OTU relative abundance based on whether individuals did not have tumors, had an adenoma, or had a carcinoma, SCFA concentrations did not provide increased model accuracy for tumor classification [Figure 3-4 & S1]. Instead, SCFA concentrations provided similar information to what specific OTUs were already providing to the tumor classification models [Figure 4]. Additionally, when models using OTU relative abundance to classify SCFA concentrations were assessed, the OTUs that classified to *Clostridiales*, *Lachnospiraceae*, and *Ruminococcaceae* were not the same as the OTUs that classified to  these taxa in the tumor models [Figure 4-5]. Collectively, our observations suggest that resident taxa from *Clostridiales*, *Lachnospiraceae*, and *Ruminococcaceae*, that are different between individuals without tumors and patients with adenomas or carinomas, are not that same as those involved with SCFA production.    

Although SCFAs have been shown to be anti-tumorigenic, most of these studies have been performed in model systems. Additionally, many of the *in vivo* studies investigate proxies such as fiber supplementation rather than SCFAs directly [@Holscher2017].  

Much of the previous research on SCFA benefit to human CRC has been illustrated in model systems [@test_OKeefe2016]. Many SCFAs are produced through the breakdown of fiber [@Holscher2017] and a recent study in mice found that fiber supplementation increased SCFA concentrations and decreased tumor burden [@Bishehsari2018]. Additionally, SCFAs such as butyrate can inhibit tumor growth in *in vitro* experiments [@Encarnao2018]. Yet, observations in humans has been mixed. Previous case/control studies found associations with lower SCFA concentrations in individuals with carcinoma tumors [@Ohigashi2013]. However, individual randomized controlled trials and a recent meta-analysis on fiber supplementation to prevent tumor recurrence has found no benefit [@Schatzkin2000; @Yao2017]. Our results align with what has been reported in randomized-controlled trials, that SCFAs do not provide general protection against colorectal tumors. It is possible though that there are specific instances where SCFAs may be beneficial. 

One limitation of current research into the effect of SCFAs in CRC has been that all tumors are treated as the same type. However, there are known differences in the types of mutations that occur [@Fearon1990] and treating all tumors as equal may actually hide any benefit that could be found in specific subsets of individuals. Similar to the idea of specific immunotherapy for specific tumors [@Thomas2016], SCFAs may have beneficial effects for specific types of colorectal tumors. Future research will need to test if this is a valid hypothesis. Another limitation is that a fecal sample may not be an ideal type of biospecimen and that the effect SCFAs have on tumorigenesis is only detected in the colon. However, most *in vivo* studies as well as human studies have used fecal material in their analysis [@Ohigashi2013; @Bishehsari2018]. Additionally, studies that measure SCFA changes after fiber supplementation use fecal material to track these responses [@scfa_measures_venkataraman2016]. Although there are limitations with the current research on SCFAs and colorectal tumors, our observations along with the randomized controlled trials on fiber supplementation in tumor recurrence [@Yao2017] provide evidence that these specific metabolites may not be protective. Yet, taxa that are associated with SCFA production are consistently higher in indivdiuals without colon tumors than patients with carcinomas [@Shah2017; @normalization_Sze2017; @meta_analysis_crc_Sze2018].      

The potential protection against colorectal cancers may not be from SCFAs even though taxa associated with their production are higher in individuals without tumors versus patients with carcinomas [@Shah2017; @normalization_Sze2017; @meta_analysis_crc_Sze2018]. Protection could be via a different pathway and by extension other metabolites that have not been extensively studied. Alternatively, protection may not occur via a metabolite but instead through niche exclusion of mouth-associated microbes (e.g. *Fusobacterium*, *Porphyromonas*, *Parvimonas*, *Peptostreptococcus* [@Zeller2014; @Baxter2016; @Kostic2011]). The idea of niche exclusion is similar to how the community protects against *Clostridium difficile* infection [@Theriot2015] with chronic inflammation replacing the role of antibiotics. Although we did not find lower concentrations of SCFAs associated with colorectal tumors, we think that there are many exciting new avenues to explore because of these results.       


\newpage

## Conclusions

Our observations found no difference in SCFA concentration, their utility as a classification tool, or for genes of enzymes involved in SCFA synthesis between individuals without colon tumors and patients with either adenoma or carcinoma tumors. Although these results are different than other reports in the literature, they do align with the randomized controlled trials that have tested fiber use in preventing colorectal tumor recurrence. Overall, these results suggest that the SCFAs typically produced by resident microbes do not protect against tumor. By focusing on other types mechanisms, the identification of more promising therapeutic options for use in treating colorectal cancer may be found.


\newpage

## Materials and Methods

**Study design and sampling.** The overall protocol has been described in detail previously [@normalization_sze2017; @crc_model_baxter2016]. In brief, this study used fecal samples obtained at either a single cross-sectional time point (n=`r length(metai$sample)`) or from before (pre-) and after (post-) treatment of a patient's tumor (n=`r length(metaf$initial)`). For patients undergoing treatment for their tumor the length of time between their initial and follow up sample ranged from `r min(metaf$time)` - `r max(metaf$time)` days. Our use of treatment has been previously defined as encompassing removal of a tumor with or without chemotherapy and radiation [@normalization_sze2017]. Diagnosis of tumor was made by colonoscopic examination and histopathological review of biopsies obtained [@normalization_sze2017; @crc_model_baxter2016]. The University of Michigan Institutional Review Board approved the study and informed consent was obtained from all participants in accordance to the guidelines set out by the Helsinki Declaration. 

**Measuring specific SCFAs.** Our protocol for the measurement of acetate, butyrate, and propionate followed a previously published protocol that used a High-Performance Liquid Chromatography (HPLC) machine [@scfa_measures_venkataraman2016]. The following changes to this protocol included the use of frozen fecal samples suspended in 1ml of PBS instead of fecal suspensions in DNA Genotek OmniGut tubes, and the use of the actual weight of fecal samples instead of the average weight for SCFA concentration normalizations. These methodological changes did not affect the overall median concentrations of these SCFAs between the two studies (see Table 1 [@scfa_measures_venkataraman2016] and Figure 1 here).    

**16s rRNA gene sequencing.** The workflow and processing have been previously described [@Schloss2009; @Kozich2013; @normalization_sze2017]. The major differences from these previous reports include: the use of version 1.39.5 of the mothur software package and clustering Operational Taxonomic Units (OTUs) at 97% similarity using the OptClust algorithm [@opticlust_westcott2017].  

**Generating imputed metagenomes.** The use of PICRUSt version 1.1.2 with the recommended standard operating protocol [@Langille2013] was used. Briefly, the mothur shared file and metadata was converted into a biom formatted table using the biom convert function, the subsequent biom file was processed with the 'normalize_by_copy_number.py' function, and subsequent imputed metagenomes created using the 'predict_metagenomes.py' function. 

**Obtaining Operational Protein Families from metagenomes.** A subset of the cross-sectional group (n=`r length(metai$sample)`) containing a total of `r length(geof_samples$X1)` individuals (normal n=`r filter(geof_samples, X30 == "Healthy") %>% pull(X30) %>% length()`, adenoma n=`r filter(geof_samples, X30 == "Adenoma") %>% pull(X30) %>% length()`, and carcinoma n=`r filter(geof_samples, X30 == "Cancer") %>% pull(X30) %>% length()`) was shotgun sequenced on an Illumina HiSeq using 125 bp paired end reads and a previously described method [@Hannigan2017]. Briefly, the sequences were quality filtered and sequences aligning to the human genome were removed prior to contig assembly with MEGAHIT [@Li2015]. Open Reading Frames (ORFs) were identified using Prodigal [@Hyatt2010], counts generated using Diamond [@Buchfink2014], subsequent clustering into Operational Protein Families (OPFs) used mmseq2 [@Steinegger2017], and OPF alignment used the KEGG database [@Kanehisa2015]. 

**Pulling genes involved with SCFA synthesis.** Specific genes located near the end of the pathways involved in the synthesis of acetate, butyrate, and propionate were analyzed for any differences between individuals with normal colons and those with tumors. These genes were based on pathways from KEGG as well as previous research [@scfa_baxter2014; @Kanehisa2015] and a list can be found in the supplemental material [Table S1]. 

**Random Forest models.** The model was first trained on 80% of the data and then tested on the held out 20% (80/20 split) using the Random Forest algorithm for classification models [@randomforest_citation_2002]. This was repeated on 100 different 80/20 splits of the data to generate a reasonable range for the AUC of the model. The reported AUCs, unless otherwise specified, are for the test sets. The classification models were built to group normal versus adenoma, normal versus carcinoma, and high versus low SCFA concentrations. 

**Statistical analysis workflow.** All analysis was performed using the statistical language R [@r_citation_2017]. Generally, a Kruskal-Walis rank sum test with a Dunn's post-hoc test was used to assess differences between the groups used. Where appropriate Benjamini-Hochberg was used to correct for multiple comparisons [@benjamini_controlling_1995]. First, we assessed differences in SCFA concentrations measured by HPLC between individuals with normal colons and patients with tumors (adenoma or carcinoma). We then analyzed whether SCFA concentrations changed in patients with an adenoma or carcinoma pre- versus post-treatment. Next, we assessed whether OTUs alone or OTUs and SCFAs were better able to classify individuals with and without tumor using Random Forest models. Next, the imputed gene counts of important mediators of SCFA synthesis was tested. Additionally, the counts generated for OPFs that matched important genes involved with SCFA creation were analyzed. Finally, models to classify high or low SCFA concentration based on the median of each SCFA and 16S rRNA gene sequencing data was created using the Random Forest algorithm.

\newpage

## Acknowledgements

The authors thank the Great Lakes-New England Early Detection Research Network for providing the fecal samples that were used in this study. We would also like to thank Kwi Kim and Thomas M Schmidt for their help in running the short-chain fatty acid analysis on the High-Performance Liquid Chromatography machine at the University of Michigan. Salary support for Marc A. Sze came from the Canadian Institute of Health Research and NIH grant UL1TR002240. Salary support for Patrick D. Schloss came from NIH grants P30DK034933 and 1R01CA215574.   

\newpage

## References

<div id="refs"></div>

\newpage

**Figure 1. No change in SCFA measurements was observed between normal, adenoma, and carcinoma individuals using HPLC.** Acetate concentrations in fecal samples of individuals without colon tumors, adenomas, and carcinomas (A). Butyrate concentrations in fecal samples of individuals without colon tumors, adenomas, and carcinomas (B). Propionate concentrations in fecal samples of individuals without colon tumors, adenomas, and carcinomas (C). The black lines indicate the median SCFA concentration. Acetate concentrations in fecal samples before and after treatment for adenoma (yellow) and carcinoma (red) (D). Butyrate concentrations in fecal samples before and after treatment for adenoma (yellow) and carcinoma (red) (E). Propionate concentrations in fecal samples before and after treatment for adenoma (yellow) and carcinoma (red) (F). The black dots and lines represent the median change in SCFA concentration.

**Figure 2. No change in butyrate producing genes identified between normal, adenoma, and carcinoma individuals.** Imputed gene relative abundance of important butyrate pathway genes using PICRUSt (A). Counts per million (corrected for size and number of contigs in an OPF) for the Butyrate Kinase gene (B). The other butyrate pathway genes from the PICRUSt analysis did not align to any of the OPFs in the metagenome analysis.

**Figure 3. Patients with adenomas had the higest number of significant negative correlations between OTU relative abundance and SCFA concentration.** Colors denote the family or lowest taxonomic ID that an OTU classified to. Fewer significant positive correlations were observed overall. Additionally, the differences in the number of significant positive correlations between patients with adenomas versus individuals without tumors (normal) and patients with carcinomas was not as pronounced as the number of significant negative correlations.  

**Figure 4. SCFA concentrations do not improve OTU-based Random Forest models.** The area under the curve of 100 different 80/20 split OTU-based normal versus adenoma 10-fold CV models with and without SCFAs (A). The top 10 most important OTUs or SCFAs in the SCFA and OTU adenoma model (B). The top 10 most important OTUs in the OTU adenoma model (C). The area under the curve of 100 different 80/20 OTU-based normal versus carcinoma 10-fold CV models with and without SCFAs (D). The top 10 most important OTUs or SCFAs in the SCFA and OTU carcinoma model (E). The top 10 most important OTUs in the OTU carcinoma model (F). For (A) and (D) the black line represents the median AUC. The dotted line highlights an AUC of 0.5.  

  
**Figure 5. OTU-based regression Random Forest models of SCFA concentrations.** The train and test correlation between actual and predicted values from 100 different 80/20 split OTU-based models with 10-fold CV using regression Random Forest (A). The model accuracy of predicted SCFA concentrations differed between individuals without tumors, patients with adenomas, and patients with carcinomas. Generally, patients with carcinomas had predicted concentrations closest to their actual measured concentration (B). The top 10 OTUs based on mean decrease in accuracy (MDA) for each SCFA model, colored by their lowest taxonomic identification (C).



\newpage

**Figure S1. Patients with adenomas had the higest number of significant differences in OTU relative abundance between high/low SCFA groups.** Colors denote the family or lowest taxonomic ID that an OTU classified to. Fewer significant OTUs were observed in individuals without tumors (normal) and patients with carcinomas versus patients with adenomas.

**Figure S2. OTU-based classification Random Forest models of high/low SCFA groups based on overall SCFA median concentration.** The train and test results of 100 different 80/20 split OTU-based models with 10-fold CV based on higher or lower than the median SCFA concentration using classification Random Forest (A). The model accuracy of predicted high/low SCFA groups differed between individuals without tumors, patients with adenomas, and patients with carcinomas. Patients with adenomas consistently had the best classification accuracy (B). The top 10 OTUs based on mean decrease in accuracy (MDA) for each SCFA model, colored by their lowest taxonomic identification (C).




