---
title: "Revisiting Short-Chain Fatty Acids and the Microbiota in Colorectal Cancer"
bibliography: references.bib
output:
  pdf_document:
    keep_tex: true
    includes:
      in_header: header.tex
csl: mbio.csl 
fontsize: 11pt
geometry: margin=1.0in
---


```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE, message=FALSE, warning=FALSE}
source('../code/functions.R')

loadLibs(c("tidyverse", "knitr", "rmarkdown"))
```


```{r rdata_tables, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

# Read in meta data
metaf <- read_csv("../data/raw/metadata/good_metaf_final.csv")
metai <- read_csv("../data/raw/metadata/metaI_final.csv")
geof_samples <- read_tsv("../data/raw/metadata/303030_subset/NexteraXT003Map.tsv", 
                         col_names = F) %>% filter(X22 != "Negative")

# Read in analysis of SCFA by HPLC
hplc_groups_scfa_kruskal <- read_csv("../data/process/tables/acetate_kruskal_crc_groups.csv") %>% 
  mutate(scfa = "acetate") %>% slice(1) %>% 
  bind_rows(
    read_csv("../data/process/tables/butyrate_kruskal_crc_groups.csv") %>% 
      mutate(scfa = "butyrate") %>% slice(1), 
    read_csv("../data/process/tables/propionate_kruskal_crc_groups.csv") %>% 
      mutate(scfa = "propionate") %>% slice(1)) %>% 
  mutate(bh = p.adjust(pvalue, method = "BH"))

hplc_treat_scfa_kruskal <- read_csv("../data/process/tables/scfa_treatment_comparison_pvalues.csv") %>% 
  filter(scfa != "isobutyrate")

# Load in Random Forest with and without SCFA summary data
disease_rf <- read_csv("../data/process/tables/rf_model_ttest_comparisons.csv")

# Load in data for select genes in SCFA pathways
picrust_select <- read_csv("../data/process/specific_scfa_kruskal_picrust_summary.csv")
opf_select <- read_csv("../data/process/select_scfa_opf_kruskal_summary.csv")

```


\vspace{35mm}

Running title: SCFAs and Colorectal Cancer

\vspace{35mm}


Marc A Sze${^1}$, Nicholas A Lesniak${^1}$, Mack T Ruffin IV${^2}$, Patrick D. Schloss${^1}$${^\dagger}$

\vspace{40mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI 48109

$2$ Department of Family Medicine and Community Medicine, Penn State Hershey Medical Center, Hershey, PA


\newpage
\linenumbers


## Abstract


\newpage

## Introduction


\newpage


## Results 

**Decreased SCFA concentrations are not associated with tumors.** We used frozen fecal samples from `r length(metai$sample)` individuals and HPLC to measure acetate, butyrate, and propionate concentrations at a cross-sectional point in time. There was no difference between individuals with normal colons (n=`r length(filter(metai, dx == "normal") %>% pull(sample))`) and patients with either an adenoma (n=`r length(filter(metai, dx == "adenoma") %>% pull(sample))`) or carcinoma (n=`r length(filter(metai, dx == "cancer") %>% pull(sample))`) for any of the SCFAs measured after multiple comparison correction (P-value `r ifelse(min(pull(hplc_groups_scfa_kruskal, bh)) > 0.15, invisible(">"), invisible("<"))` 0.15) [Figure 1A - 1C]. We next measured the concentration of SCFAs in `r length(metaf$initial)` patients with an adenoma (n=`r length(filter(metaf, dx == "adenoma") %>% pull(initial))`) or carcinoma (n=`r length(filter(metaf, dx == "cancer") %>% pull(initial))`) in which we had pre- and post-treatment fecal samples. Although there was a general trend for increasing acetate, butyrate, and propionate concentrations after treatment for tumors, there was no significant difference pre- and post-treatment for patients with adenoma (P-value `r ifelse(min(filter(hplc_treat_scfa_kruskal, disease == "adenoma") %>% pull(bh)) > 0.20, invisible(">"), invisible("<"))` 0.20) or carcinoma (P-value `r ifelse(min(filter(hplc_treat_scfa_kruskal, disease == "carcinoma") %>% pull(bh)) > 0.80, invisible(">"), invisible("<"))` 0.80) [Figure 1D - 1F]. Even though there was no change in SCFA concentrations between individuals with normal colons and those with tumors, this information could still be important to help classify disease.

**Random Forest models with SCFA concentrations do not classify tumors better.** Using the Random Forest algorithm we built models to classify normal versus adenoma and normal versus carcinoma with OTUs or OTUs and SCFA concentrations. For both adenoma and carcinoma models, there was no difference between the median AUC of models with or without SCFA concentrations (P-value `r ifelse(min(pull(disease_rf, bh)) > 0.05, invisible(">"), invisible("<"))` 0.05) [Figure 2]. Although including SCFA concentrations did not add extra information for classification of disease using Random Forest models, it is possible that the genes for enzymes involved in SCFA synthesis may change due to disease status; where a smaller number of microbes are responsible for the observed SCFA concentrations.

**Changes in genes for enzymes involved in SCFA synthesis are not associated with tumors.** Using a list of specific genes that are important for the synthesis of SCFAs [Table S1], we looked for differences in gene abundance based on individuals having normal colons, adenomas, or carcinomas. First, using imputed gene relative abundance based on 16S rRNA gene sequencing we found no difference in any of the genes inolved with acetate, butyrate, or propionate synthesis (P-value `r ifelse(min(pull(picrust_select, bh)) > 0.90, invisible(">"), invisible("<"))` 0.90) [Table S2]. This similarity between groups is highlighted by visualizing genes important in butyrate synthesis [Figure 3A]. Next, we took a subset of these `r length(metai$sample)` fecal samples (n=`r length(geof_samples$X1)`) and performed metagenomic sequencing to confirm these results. Like the imputed gene results, metagenomic analysis found that there was no difference in any of the genes involved in SCFA synthesis between individuals with normal colons (n=`r length(filter(geof_samples, X30 == "Healthy") %>% pull(X2))`) and patients with adenoma (n=`r length(filter(geof_samples, X30 == "Adenoma") %>% pull(X2))`) or carcinoma (n=`r length(filter(geof_samples, X30 == "Cancer") %>% pull(X2))`) (P-value `r ifelse(min(pull(opf_select, bh)) > 0.70, invisible(">"), invisible("<"))` 0.70) [Table S3]. This lack of difference is highlighted when we visualize the results for butyrate kinase [Figure 3B]. These observations provide evidence that the gene content does not change due to tumors. Although these results suggest that SCFA concentrations do not change due to tumor, there may be some errors present within our results that may account for the lack of associations.

**Expected taxa are associated with higher SCFA concentrations regardless of tumor status.** Using OTU data we built Random Forest models to classify higher than median and lower than median SCFA concentrations. Overall, OTU data had a reasonable ability to classify high and low SCFA concentrations [Figure S1A]. However, these models tended to be overfit, suggesting that rarer taxa may be important for this classification [Figure S1A]. The most important OTUs to these models (assessed with mean decrease in accuracy (MDA)) were to taxa that are normally associated with SCFA production [Figure S1B]. These results highlight that there are no immediately apparent errors in our data because SCFA concentrations are associated with taxa known to produce acetate, butyrate, and propionate. Additionally, OTUs associated with these taxa are the most important to models that can classify high and low SCFA concentrations. Overall, our results do not support the hypothesis that lower SCFA concentrations are associated with colorectal cancer.      

\newpage

## Discussion

The observations from this study do not support the hypothesis that SCFA concentrations are lower in individuals with tumors. Whether we directly measured the SCFA concentration or investigate genes associated with their production no difference could be identified [Figure 1 & 3]. Further, no benefit could be found in using the concentrations to help classify individuals with and without tumors [Figure 2]. These observations are in stark contrast to some of the previous literature.

Much of the previous literature on SCFA benefit to human colorectal cancer has been illustrated in model systems [@test_OKeefe2016]. Many SCFAs are produced through the breakdown of fiber [@Holscher2017]. A recent study in mice found that fiber supplementation increased SCFA concentration and decreased tumor burden [@Bishehsari2018]. Additionally, SCFAs such as butyrate have been found to inhibit tumor growth in *in vitro* experiments [@Encarnao2018]. Yet, observations in humans has been mixed. Previous case/control studies have suggested that SCFAs are lower in individuals with carcinoma tumors [@Ohigashi2013]. However, individual studies and meta-analysis on fiber supplementation to prevent tumor recurrence have found no benefit [@Schatzkin2000; @Yao2017]. Our results align with what has been reported in randomized-controlled trials that SCFAs do not protect against all types of colorectal tumors. It is possible though that there are specific instances where SCFAs may be beneficial. 

One limitation of ours and others research into the effect of SCFAs in colorectal cancer has been that all tumors are treated as the same type. However, there are known differences in the types of mutations that occur [] and treating all tumors as equal may actually hide any benefit that could be found in specific subsets of individuals. Similar to the idea of specific immunotherapy for specific tumors [], SCFAs may have beneficial effects for specific types of colorectal cancers. Future research will need to test if this is a valid hypothesis. Although there are limitations with the current research on SCFAs and colorectal tumors, evidence is building that these specific metabolites may not be protective. Yet, taxa that are associated with SCFA production are consistently higher in normal colons than those with carcinomas [].      

The potential protection against colorectal cancers may not be from SCFAs even though taxa associated with their production are higher in individuals with normal colons versus patients with carcinomas []. Protection could be via a different pathway and other metabolites that have not been extensively studied. Alternatively, protection may not occur via a metabolite but instead occur through niche exclusion of mouth-associated microbes (e.g. *Fusobacterium*, *Porphyromonas*, *Parvimonas*, *Peptostreptococcus* []). The idea of niche exclusion is similar to how the community protects against *Clostridium difficile* infection with chronic inflammation replacing the role of antibiotics. Although we did not find lower concentrations of SCFAs associated with colorectal tumors, we think that there are many exciting new avenues to explore because of these observations.       








\newpage

## Conclusions

\newpage

## Materials and Methods

**Study design and sampling.** The overall protocol has been described in detail previously [@normalization_sze2017; @crc_model_baxter2016]. In brief, this study used fecal samples obtained at either a single cross-sectional time point (n=`r length(metai$sample)`) or from before (pre-) and after (post-) treatment for their tumor (n=`r length(metaf$initial)`). For patients undergoing treatment for their tumor the length of time between their initial and follow up sample ranged from `r min(metaf$time)` - `r max(metaf$time)` days. Our use of treatment has been previously defined as encompassing removal of a tumor with or without chemotherapy and radiation [@normalization_sze2017]. Diagnosis of tumor was made by colonoscopic examination and histopathological review of biopsies obtained [@normalization_sze2017; @crc_model_baxter2016]. The University of Michigan Institutional Review Board approved the study and informed consent was obtained from all participants in accordance to the guidelines set out by the Helsinki Decleration. 

**Measuring specific SCFAs.** Our protocol for the measurement of acetate, butyrate, and propionate followed a previously published protocol [@scfa_measures_venkataraman2016]. The following changes to this protocol included the use of frozen fecal samples suspended in 1ml of PBS instead of fecal suspensions in DNA Genotek OmniGut tubes, and the use of the acutal weight of fecal samples instead of the average weight for SCFA concentration normalizations. These changes did not affect the overall median concentrations of these SCFAs between the two studies (see Table 1 [@scfa_measures_venkataraman2016] and Figure 1 in this report).    

**16s rRNA gene sequencing.** The workflow and processing have been described previously [@Schloss2009; @Kozich2013; @normalization_sze2017]. The major differences from these previous reports include: the use of version 1.39.5 of the mothur software package and clustering Operational Taxonomic Units (OTUs) at 97% similarity used the OptClust algorithm [@opticlust_westcott2017].  

**Generating imputed metagenomes.** The use of PICRUSt version 1.1.2 with the recommended standard operating protocol [@Langille2013] was used. Briefly, the mothur shared file and metadata was converted into a biom formated table using the biom convert function, the subsequent biom file was processed with the normalize_by_copy_number.py function, and subsequent imputed metagenomes created using the predict_metagenomes.py function. 

**Obtaining OPFs from metagenomes.** A subset of the cross-sectional group (n=`r length(metai$sample)`) containing a total of `r length(geof_samples$X1)` individuals (normal n=`r filter(geof_samples, X30 == "Healthy") %>% pull(X30) %>% length()` normal, adenoma n=`r filter(geof_samples, X30 == "Adenoma") %>% pull(X30) %>% length()`, and carcinoma n=`r filter(geof_samples, X30 == "Cancer") %>% pull(X30) %>% length()`) was shotgun sequenced on an Illumina HiSeq using 125 bp paired end reads and a previously described method [@Hannigan2017]. Briefly, the sequences were quality filtered and sequences aligning to the human genome were removed prior to contig assembly with MEGAHIT [@Li2015]. Open Reading Frames (ORFs) were identified using Prodigal [@Hyatt2010], counts generated using Diamond [@Buchfink2014], subsequent clustering into Operational Protein Families (OPFs) used mmseq2 [@Steinegger2017], and OPF alignment used the KEGG database [@Kanehisa2015]. 

**Pulling genes involved with SCFA synthesis.** Specific genes located near the end of the pathways involved in the synthesis of acetate, butyrate, and propionate were analyzed for any differences between individuals with normal colons and those with tumors. These genes were based on pathways from KEGG as well as previous research [@scfa_baxter2014; @Kanehisa2015] and a list can be found in the supplemental material [Table S1]. 

**Random Forest Models.** The model was first trained on 80% of the data and then tested on the held out 20% (80/20 split) using the Random Forest algorithm for both classification and regression models [@randomforest_citation_2002]. This was repeated on 100 differen 80/20 splits of the data to generate a reasonable range for the AUC of the model. The reported AUCs, unless otherwise specified, are for the test sets. Classification models were built to group normal versus adenoma and normal versus carcinoma or high versus low SCFA concentrations. 

**Statistical analysis workflow.** All analysis was performed using the statistical language R [@r_citation_2017]. Generally, differences between the different disease groups used a Kruskal-Walis rank sum test with a Dunn's post-hoc test. Where appropriate Benjamini-Hochberg was used to correct for multiple comparisons [@benjamini_controlling_1995]. First, we assessed differences in SCFA concentrations measured by HPLC between individuals with normal colons and patients with tumors (adenoma or carcinoma). We then analyzed whether SCFA concentrations changed in patients with an adenoma or carcinoma pre- versus post-treatment. Next, we assessed whether OTUs alone or OTUs and SCFAs were better able to classify individuals with and without tumor using Random Forest models. Next, the imputed gene counts of important mediators of SCFA creation were tested. Finally, the counts generated for OPFs that matched important genes involved with SCFA creation were analyzed. Finally, models to classify high or low SCFA concentration based on 16S rRNA gene sequencing data were created using the Random Forest algorithm.

\newpage

## Acknowledgements

The authors thank the Great Lakes-New England Early Detection Research Network for providing the fecal samples that were used in this study. We would also like to thank Kwi Kim and Thomas M Schmidt for their help in running the short-chian fatty acid analysis on the High-Performace Liquid Chromatography machine at the University of Michigan. Salary support for Marc A. Sze came from the Canadian Institute of Health Research and NIH grant UL1TR002240. Salary support for Patrick D. Schloss came from NIH grants P30DK034933 and 1R01CA215574.   

\newpage

## References

<div id="refs"></div>

\newpage

**Figure 1. Using HPLC no change in SCFA measurements was observed between normal, adenoma, and carcinoma individuals.** Acetate concentrations in fecal samples of individuals with normal colons, adenomas, and carcinomas (A). Butyrate concentrations in fecal samples of individuals with normal colons, adenomas, and carcinomas (B). Propionate concentrations in fecal samples of individuals with normal colons, adenomas, and carcinomas (C). The black links indicate the median SCFA concentration. Acetate concentrations in fecal samples before and after treatment for adenoma (yellow) and carcinoma (red) (D). Butyrate concentrations in fecal samples before and after treatment for adenoma (yellow) and carcinoma (red) (E). Propionate concentrations in fecal samples before and after treatment for adenoma (yellow) and carcinoma (red) (F). The black dots and lines represent the median change in SCFA concentration.

**Figure 2. SCFAs do not improve OTU-based Random Forest models.** Difference between the area under the curve of 100 different 80/20 split OTU-based normal versus adenoma 10-fold CV models with and without SCFAs (A). Difference between the area under the curve of 100 different 80/20 OTU-based normal versus carcinoma 10-fold CV models with and without SCFAs (B). The black linke represents the median AUC. The dotted line highlights an AUC of 0.5.  

**Figure 3. No change in butyrate producing genes identified between normal, adenoma, and carcinoma individuals.** Imputed gene relative abundance of important butyrate pathway genes using PICRUSt (A). Counts per million (corrected for size and number of contigs in an OPF) for the Butyrate Kinase gene (B). The other genes from the PICRUSt analysis did not align to any of the OPFs in the metagenome analysis.  


\newpage

**Figure S1. OTU-based Random Forest models of SCFA concentrations.** Classification Random Forest train and tests of 100 different 80/20 OTU-based models with 10-fold CV based on higher or lower than the medain SCFA concentration (A). The top 10 OTUs based on mean decrease in accuracy (MDA) for each model, colored by their lowest taxonomic identification (B). Regression Random Forest train and tests of 100 different 80/20 OTU-based models with 10-fold CV based on correlation to actual SCFA concentration (C). The top 10 OTUs based on mean decrease in accuracy (MDA) for each model, colored by their lowest taxonomic identification (D).




