---
title: ''
csl: mbio.csl
fontsize: 11pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
geometry: margin=0.75in
bibliography: null
---
\pagenumbering{gobble}
```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("tidyverse", "scales", "knitr", "rmarkdown", "stringr"))

```


**Table S1: SCFA enzymes and corresponding KEGG ID analyzed.**
```{r supplemental_table1, echo=FALSE, message=FALSE, warning=FALSE}

Kegg_ID <- c("K00929", "K01034", "K01035", "K01908", "K01895", "K00932", "K19697", 
             "K18372", "K00467", "K00156", "K00925", "K01512", "K01067", "K18118", "K01026", 
             "K01905", "K22224", "K01895", "K00128", "K14085", "K00149", "K00138")

Gene_Name <- c("Butyrate Kinase", "Acetate CoA subnunit A", "Acetate CoA subunit B", "Propionyl-CoA Synthetase", 
               "Acetyl-CoA Synthetase", "Propionate Kinase", "Propionate Kinase", "Methyl Acetate Hydrolase", 
               "Lactate 2-Monooxygenase", "Pyruvate Dehydrogenase", "Acetate Kinase", "Acylphosphatase", 
               "Acetyl-CoA Hydrolase", "Succinyl-CoA:Acetate", "Propionate CoA-Transferase", 
               "Acetate---CoA Ligase subunit A", "Acetate---CoA Ligase subunit B", "Acetyl-CoA Synthetase", 
               "Aldehyde Dehydrogenase", "Aldehyde Dehydrogenase Family 7", "Aldehyde Dehydrogenase Family 9", 
               "Aldehyde Dehydrogenase")

sup_table1 <- data_frame(Kegg_ID, Gene_Name)

colnames(sup_table1) <-  c("KEGG IDs", "Enzyme Name")


kable(sup_table1, format = "markdown", align = 'c', padding = 2)


```

\newpage

**Table S2: Breakdown of Kruskal-Wallis tests using PICRUSt imputed relative abundance of select genes in SCFA pathways.**
```{r supplemental_table2, echo=FALSE, message=FALSE, warning=FALSE}
table_s2 <- read_csv("../data/process/specific_scfa_kruskal_picrust_summary.csv") %>% 
  select(kegg_ortholog, statistic, p.value, bh) %>% 
  mutate_at(vars(statistic:bh), function(x) round(x, digits = 3))

colnames(table_s2) <-  c("KEGG IDs", "Rank Sum Statistic", "P-value", "BH")

kable(table_s2, format = "markdown", align = 'c', padding = 2)
# opf_select <- read_csv("../data/process/select_scfa_opf_kruskal_summary.csv")
```


\newpage



**Table S3: Breakdown of Wilcoxson rank-sum tests for pre- versus post-treatment of tumor using PICRUSt imputed relative abundance of select genes in SCFA pathways.**
```{r supplemental_table3, echo=FALSE, message=FALSE, warning=FALSE}

table_s3 <- read_csv("../data/process/tables/specific_scfa_wilcox_picrust_treatment_summary.csv") %>% 
  select(kegg_ortholog, group, statistic, p.value, bh) %>% 
   mutate_at(vars(p.value:bh), function(x) round(x, digits = 3)) %>% 
  arrange(kegg_ortholog, group)
 

colnames(table_s3) <-  c("KEGG IDs", "Disease Group", "Test Statistic", "P-value", "BH")

kable(table_s3, format = "markdown", align = 'c', padding = 2)
```


\newpage

**Table S4: Breakdown of Kruskal-Wallis tests using counts per million from metagenomic sequencing of select genes in SCFA pathways.**
```{r supplemental_table4, echo=FALSE, message=FALSE, warning=FALSE}
table_s4 <- read_csv("../data/process/select_scfa_opf_kruskal_summary.csv") %>% 
  select(kegg_id, statistic, p.value, bh) %>% 
  mutate_at(vars(statistic:bh), function(x) round(x, digits = 3))

colnames(table_s4) <-  c("KEGG IDs", "Rank Sum Statistic", "P-value", "BH")

kable(table_s4, format = "markdown", align = 'c', padding = 2)
```


\newpage


**Table S5: Spearman tests between OTU relative abundance and SCFA concentrations.**

\scriptsize

```{r supplemental_table5, echo=FALSE, message=FALSE, warning=FALSE}

table_s5 <- read_csv("../data/process/tables/significant_reg_otu_comp_summary.csv") %>% 
  left_join(read_tsv("../data/process/final.taxonomy") %>% 
              select(-Size) %>% 
  mutate(Taxonomy = str_replace_all(Taxonomy, "\\(\\d*\\)", "")) %>% 
  separate(Taxonomy, c("kingdom", "phyla", "class", "order", "family", "genus", "species"), sep = ";") %>% 
              select(OTU, family), by = c("otu" = "OTU")) %>% 
  select(otu, family, genus, dx, scfa, estimate, p.value, bh) %>% 
  mutate_at(vars(p.value:bh), function(x) format(x, scientific = T, digits = 3)) %>% 
  mutate(estimate = round(estimate, digits = 3), 
         family = str_replace_all(family, "_unclassified", ""), 
         family = str_replace_all(family, "_", " "), 
         genus = str_replace_all(family, "_unclassified", ""), 
         genus = str_replace_all(family, "_", " ")) %>% 
  arrange(scfa, dx, bh)

colnames(table_s5) <-  c("OTU", "Family", "Genus", "Group", "SCFA", "Rho", "P-value", "BH")

kable(table_s5, format = "markdown", align = 'c', padding = 2)

```

\normalsize

\newpage

**Table S6: Wilcoxson tests between OTU relative abundance and high or low SCFA concentrations based on the overall median.**

\scriptsize

```{r supplemental_table6, echo=FALSE, message=FALSE, warning=FALSE}

table_s6 <- read_csv("../data/process/tables/significant_class_otu_comp_summary.csv") %>% 
  left_join(read_tsv("../data/process/final.taxonomy") %>% 
              select(-Size) %>% 
  mutate(Taxonomy = str_replace_all(Taxonomy, "\\(\\d*\\)", "")) %>% 
  separate(Taxonomy, c("kingdom", "phyla", "class", "order", "family", "genus", "species"), sep = ";") %>% 
              select(OTU, family), by = c("otu" = "OTU")) %>% 
  select(otu, family, genus, dx, scfa, p.value, bh) %>% 
  mutate_at(vars(p.value:bh), function(x) format(x, scientific = T, digits = 3)) %>% 
  mutate(family = str_replace_all(family, "_unclassified", ""), 
         family = str_replace_all(family, "_", " "), 
         genus = str_replace_all(family, "_unclassified", ""), 
         genus = str_replace_all(family, "_", " ")) %>% 
  arrange(scfa, dx, bh)

colnames(table_s6) <-  c("OTU", "Family", "Genus", "Group", "SCFA", "P-value", "BH")

kable(table_s6, format = "markdown", align = 'c', padding = 2)

```

\normalsize


















